<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0043)http://www.ncbi.nlm.nih.gov/books/NBK26887/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="">
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books"><meta name="ncbi_pdid" content="book-part"><meta name="ncbi_acc" content="NBK26887"><meta name="ncbi_domain" content="mboc4"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/NBK26887/"><meta name="ncbi_app" content="bookshelf">
        <!-- Logger end -->
        
        <title>From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/jig.min.js"></script><style type="text/css"></style><link rel="stylesheet" type="text/css" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/jig.core.theme.min.css"><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/jquery.ui.widgets.min.js"></script><link type="text/css" rel="stylesheet" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/jquery.ui.widgets.min.css">

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE,NOIMAGEINDEX"><meta name="author" content="Bruce Alberts, Alexander Johnson, Julian Lewis, Martin Raff, Keith Roberts, Peter Walter"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="DC.Title" content="From DNA to RNA"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Garland Science"><meta name="DC.Contributor" content="Bruce Alberts"><meta name="DC.Contributor" content="Alexander Johnson"><meta name="DC.Contributor" content="Julian Lewis"><meta name="DC.Contributor" content="Martin Raff"><meta name="DC.Contributor" content="Keith Roberts"><meta name="DC.Contributor" content="Peter Walter"><meta name="DC.Date" content="2002"><meta name="DC.Identifier" content="http://www.ncbi.nlm.nih.gov/books/NBK26887/"><meta name="DC.Language" content="en"><meta name="bk-non-canon-loc" content="/books/n/mboc4/A975/"><meta name="bk-srch-limit" content="mboc4[book]"><link rel="canonical" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf.htm"><link rel="stylesheet" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/figpopup.css" type="text/css" media="screen"><link rel="stylesheet" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/books.css" type="text/css"><link rel="stylesheet" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/books_print.css" type="text/css" media="print"><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .body-content h2 {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important}</style><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/common.min.js"> </script><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/book-toc.js"> </script><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/books.js"> </script>
        <!-- Page meta end -->
    <link rel="shortcut icon" href="http://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="396E6EEB20DDDA410000000000888C4C"><link type="text/css" rel="stylesheet" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/3395586.css"><link type="text/css" rel="stylesheet" href="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/3882866.css" media="print"><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('/stat?jsevent=print&ncbi_app=bookshelf&ncbi_db=books&ncbi_pcid=%2FNBK26887%2F&ncbi_pdid=book-part&ncbi_phid=396E6EEB20DDDA410000000000888C4C'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="http://www.ncbi.nlm.nih.gov/corehtml/query/static/unsupported-browser.html#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="http://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="http://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="http://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="http://www.ncbi.nlm.nih.gov/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="http://www.ncbi.nlm.nih.gov/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="http://www.ncbi.nlm.nih.gov/account/?back_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK26887%2F" id="sign_in">Sign in to NCBI</a><a href="http://www.ncbi.nlm.nih.gov/account/signout/?back_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK26887%2F" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="http://www.ncbi.nlm.nih.gov/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <form method="get" action="http://www.ncbi.nlm.nih.gov/books/" id="entrez-search-form">
  <input type="hidden" id="ncbisearchbar-app-name" value="bookshelf">
  <div class="search">
    <label for="term" class="offscreen_noflow">Search term</label>
    <label for="entrez-search-db" class="offscreen_noflow">Search database</label>
    <div class="search_form">
      <select id="entrez-search-db" name="resource">
        <option value="gquery">All Databases</option>
        <option value="pubmed">PubMed</option>
        <option value="protein">Protein</option>
        <option value="nuccore">Nucleotide</option>
        <option value="nucgss">GSS</option>
        <option value="nucest">EST</option>
        <option value="structure">Structure</option>
        <option value="genome">Genome</option>
        <option value="assembly">Assembly</option>
        <option value="bioproject">BioProject</option>
        <option value="biosample">BioSample</option>
        <option value="books" selected="selected">Books</option>
        <option value="cdd">Conserved Domains</option>
        <option value="clinvar">ClinVar</option>
        <option value="clone">Clone</option>
        <option value="gap">dbGaP</option>
        <option value="dbvar">DbVar</option>
        <option value="epigenomics">Epigenomics</option>
        <option value="gene">Gene</option>
        <option value="gds">GEO DataSets</option>
        <option value="geoprofiles">GEO Profiles</option>
        <option value="homologene">HomoloGene</option>
        <option value="medgen">MedGen</option>
        <option value="mesh">MeSH</option>
        <option value="ncbisearch">NCBI Web Site</option>
        <option value="nlmcatalog">NLM Catalog</option>
        <option value="omia">OMIA</option>
        <option value="omim">OMIM</option>
        <option value="pmc">PMC</option>
        <option value="popset">PopSet</option>
        <option value="probe">Probe</option>
        <option value="proteinclusters">Protein Clusters</option>
        <option value="pcassay">PubChem BioAssay</option>
        <option value="biosystems">BioSystems</option>
        <option value="pccompound">PubChem Compound</option>
        <option value="pcsubstance">PubChem Substance</option>
        <option value="pubmedhealth">PubMed Health</option>
        <option value="snp">SNP</option>
        <option value="sra">SRA</option>
        <option value="taxonomy">Taxonomy</option>
        <option value="toolkit">ToolKit</option>
        <option value="toolkitall">ToolKitAll</option>
        <option value="toolkitbook">ToolKitBook</option>
        <option value="unigene">UniGene</option>
        <option value="unists">UniSTS</option>
      </select>
      <div class="nowrap">
        <div class="nowrap">
          <div class="jig-ncbiclearbutton-wrap"><input name="term" type="text" id="term" value="" class="jig-ncbiautofocus jig-ncbiclearbutton jig-ncbiautocomplete" autocomplete="off" config="disableUrl:&#39;?session$autocomplete=false&#39;" title="Use up and down arrows to choose an item from the autocomplete." aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" style="visibility: hidden;"><img src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/clear.png" alt="Clear input"></a></div>
        </div>
        <button type="submit" id="search" class="button_search nowrap">Search</button>
      </div>
    </div>
    <ul class="searchlinks inline_list">
      <li>
                        <a href="http://www.ncbi.nlm.nih.gov/books/browse/">Browse Titles</a>
                    </li>
      <li>
                        <a href="http://www.ncbi.nlm.nih.gov/books/limits/">Limits</a>
                    </li>
      <li>
                        <a href="http://www.ncbi.nlm.nih.gov/books/advanced/">Advanced</a>
                    </li>
      <li class="help">
                        <a href="http://www.ncbi.nlm.nih.gov/books/NBK3833/">Help</a>
                    </li>
    </ul>
  </div>
</form>

</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Alberts B, Johnson A, Lewis J, et al. Molecular Biology of the Cell. 4th edition. New York: Garland Science; 2002. </p></div><div class="messagearea bk_noprnt" style="margin-bottom:1.3846em "><ul class="messages"><li class="info icon"><span class="icon">By agreement with the publisher, this book is accessible by the search feature, but cannot be browsed.</span></li></ul></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="Table of Contents Page" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/TOC/"><img class="source-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/th-mboc4-lrg.png" alt="Cover of Molecular Biology of the Cell" height="100px" width="80px"></a><div class="icnblk_cntnt eight_col"><h2>Molecular Biology of the Cell. 4th edition.</h2><a class="jig-ncbitoggler ui-widget ui-ncbitoggler" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/__NBK26887_dtls__" aria-disabled="false" role="button" aria-expanded="false"><span class="ui-ncbitoggler-master-text">Show details</span><span class="ui-icon ui-icon-triangle-1-e"></span></a><div class="ui-helper-reset" aria-live="assertive"><div style="display:none" id="__NBK26887_dtls__" class="ui-ncbitoggler ui-ncbitoggler-slave" aria-hidden="true"><div>Alberts B, Johnson A, Lewis J, et al.</div><div>New York: <a href="http://www.garlandscience.com/textbooks/0815341059.asp" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">Garland Science</a>; 2002.</div></div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"></li></ul></div><form method="get" action="http://www.ncbi.nlm.nih.gov/books/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><div class="jig-ncbiclearbutton-wrap"><input type="text" class="jig-ncbiclearbutton" title="Search this book" id="bk_term" name="term" value=""><a class="reset" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" style="visibility: hidden;"><img src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/clear.png" alt="Clear input"></a></div> <input type="submit" class="jig-ncbibutton ui-state-default ui-corner-all ui-ncbibutton ui-ncbibutton-gray" value="Search this book" submit="false" style="padding: 0.1em 0.4em;" role="button" aria-disabled="false"></div></form></div></div></div></div>
            <div class="main-content"><div class="meta-content"><h1><span class="title">From DNA to RNA</span></h1></div><div class="body-content whole_rhythm jig-ncbiinpagenav" data-jigconfig="allHeadingLevels: [&#39;h2&#39;],smoothScroll: false"><p>Transcription and translation are the means by which cells read out, or express, the genetic instructions in their genes. Because many identical <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> copies can be made from the same <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>, and each RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> can direct the synthesis of many identical <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> molecules, cells can synthesize a large amount of protein rapidly when necessary. But each gene can also be transcribed and translated with a different efficiency, allowing the cell to make vast quantities of some proteins and tiny quantities of others (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A976/?report=objectonly" target="object" rid-figpopup="figA976" rid-ob="figobA976">Figure 6-3</a>). Moreover, as we see in the next chapter, a cell can change (or regulate) the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5163/" role="button" aria-expanded="false" aria-haspopup="true">expression</a> of each of its genes according to the needs of the moment—most obviously by controlling the production of its RNA.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA976" co-legend-rid="figlgndA976"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A976/?report=objectonly" target="object" title="Figure 6-3" class="img_link icnblk_img figpopup" rid-figpopup="figA976" rid-ob="figobA976"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f3.gif" src-large="/books/NBK26887/bin/ch6f3.jpg" alt="Figure 6-3. Genes can be expressed with different efficiencies."></a><div class="icnblk_cntnt" id="figlgndA976"><h4 id="A976"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A976/?report=objectonly" target="object" rid-ob="figobA976">Figure 6-3</a></h4><p class="float-caption no_bottom_margin">Genes can be expressed with different efficiencies. Gene A is transcribed and translated much more efficiently than gene B. This allows the amount of protein A in the cell to be much greater than that of protein B. </p></div></div><div id="A977"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 id="_A977_" class="ui-helper-clearfix">Portions of DNA Sequence Are Transcribed into RNA</h2><p>The first step a cell takes in reading out a needed part of its genetic instructions is to copy a particular portion of its <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence—a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>—into an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> nucleotide sequence. The information in RNA, although copied into another chemical form, is still written in essentially the same language as it is in DNA—the language of a nucleotide sequence. Hence the name <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5880/" class="def" role="button" aria-expanded="false" aria-haspopup="true">transcription</a>.</p><p>Like <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a>, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> is a linear <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5655/" role="button" aria-expanded="false" aria-haspopup="true">polymer</a> made of four different types of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> subunits linked together by phosphodiester bonds (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A978/?report=objectonly" target="object" rid-figpopup="figA978" rid-ob="figobA978">Figure 6-4</a>). It differs from DNA chemically in two respects: (1) the nucleotides in RNA are <i>ribonucleotides</i>—that is, they contain the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5842/" role="button" aria-expanded="false" aria-haspopup="true">sugar</a> ribose (hence the name <i>ribo</i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5561/" role="button" aria-expanded="false" aria-haspopup="true">nucleic acid</a>) rather than deoxyribose; (2) although, like DNA, RNA contains the bases adenine (A), guanine (<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/" role="button" aria-expanded="false" aria-haspopup="true">G</a>), and cytosine (C), it contains the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a> uracil (U) instead of the thymine (T) in DNA. Since U, like T, can base-pair by hydrogen-bonding with A (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A979/?report=objectonly" target="object" rid-figpopup="figA979" rid-ob="figobA979">Figure 6-5</a>), the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/" role="button" aria-expanded="false" aria-haspopup="true">complementary</a> base-pairing properties described for DNA in Chapters 4 and 5 apply also to RNA (in RNA, G pairs with C, and A pairs with U). It is not uncommon, however, to find other types of base pairs in RNA: for example, G pairing with U occasionally.


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA978" co-legend-rid="figlgndA978"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A978/?report=objectonly" target="object" title="Figure 6-4" class="img_link icnblk_img figpopup" rid-figpopup="figA978" rid-ob="figobA978"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f4.gif" src-large="/books/NBK26887/bin/ch6f4.jpg" alt="Figure 6-4. The chemical structure of RNA."></a><div class="icnblk_cntnt" id="figlgndA978"><h4 id="A978"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A978/?report=objectonly" target="object" rid-ob="figobA978">Figure 6-4</a></h4><p class="float-caption no_bottom_margin">The chemical structure of RNA. (A) RNA contains the sugar ribose, which differs from deoxyribose, the sugar used in DNA, by the presence of an additional -OH group. (B) RNA contains the base uracil, which differs from thymine, the equivalent base in DNA, <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A978/?report=objectonly" target="object" rid-ob="figobA978">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA979" co-legend-rid="figlgndA979"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A979/?report=objectonly" target="object" title="Figure 6-5" class="img_link icnblk_img figpopup" rid-figpopup="figA979" rid-ob="figobA979"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f5.gif" src-large="/books/NBK26887/bin/ch6f5.jpg" alt="Figure 6-5. Uracil forms base pairs with adenine."></a><div class="icnblk_cntnt" id="figlgndA979"><h4 id="A979"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A979/?report=objectonly" target="object" rid-ob="figobA979">Figure 6-5</a></h4><p class="float-caption no_bottom_margin">Uracil forms base pairs with adenine. The absence of a methyl group in U has no effect on base-pairing; thus, U-A base pairs closely resemble T-A base pairs (see Figure 4-4). </p></div></div><p>Despite these small chemical differences, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> differ quite dramatically in overall structure. Whereas DNA always occurs in cells as a double-stranded helix, RNA is single-stranded. RNA chains therefore fold up into a variety of shapes, just as a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/" role="button" aria-expanded="false" aria-haspopup="true">polypeptide</a> chain folds up to form the final shape of a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A980/?report=objectonly" target="object" rid-figpopup="figA980" rid-ob="figobA980">Figure 6-6</a>). As we see later in this chapter, the ability to fold into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> three-dimensional shapes allows some RNA molecules to have structural and catalytic functions.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA980" co-legend-rid="figlgndA980"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A980/?report=objectonly" target="object" title="Figure 6-6" class="img_link icnblk_img figpopup" rid-figpopup="figA980" rid-ob="figobA980"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f6.gif" src-large="/books/NBK26887/bin/ch6f6.jpg" alt="Figure 6-6. RNA can fold into specific structures."></a><div class="icnblk_cntnt" id="figlgndA980"><h4 id="A980"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A980/?report=objectonly" target="object" rid-ob="figobA980">Figure 6-6</a></h4><p class="float-caption no_bottom_margin">RNA can fold into specific structures. RNA is largely single-stranded, but it often contains short stretches of nucleotides that can form conventional base-pairs with complementary sequences found elsewhere on the same molecule. These interactions, along <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A980/?report=objectonly" target="object" rid-ob="figobA980">(more...)</a></p></div></div></div><div id="A981"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A981_" class="ui-helper-clearfix">Transcription Produces RNA Complementary to One Strand of DNA</h2><p>All of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> in a cell is made by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5098/" role="button" aria-expanded="false" aria-haspopup="true">DNA transcription</a>, a process that has certain similarities to the process of DNA replication discussed in Chapter 5. Transcription begins with the opening and unwinding of a small portion of the DNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/" role="button" aria-expanded="false" aria-haspopup="true">double helix</a> to expose the bases on each DNA strand. One of the two strands of the DNA double helix then acts as a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> for the synthesis of an RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a>. As in DNA replication, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence of the RNA chain is <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/" role="button" aria-expanded="false" aria-haspopup="true">determined</a> by the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/" role="button" aria-expanded="false" aria-haspopup="true">complementary</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pairing between incoming nucleotides and the DNA template. When a good match is made, the incoming ribonucleotide is covalently linked to the growing RNA chain in an enzymatically catalyzed <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a>. The RNA chain produced by transcription—the <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a></i>—is therefore elongated one nucleotide at a time, and it has a nucleotide sequence that is exactly complementary to the strand of DNA used as the template (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A982/?report=objectonly" target="object" rid-figpopup="figA982" rid-ob="figobA982">Figure 6-7</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA982" co-legend-rid="figlgndA982"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A982/?report=objectonly" target="object" title="Figure 6-7" class="img_link icnblk_img figpopup" rid-figpopup="figA982" rid-ob="figobA982"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f7.gif" src-large="/books/NBK26887/bin/ch6f7.jpg" alt="Figure 6-7. DNA transcription produces a single-stranded RNA molecule that is complementary to one strand of DNA."></a><div class="icnblk_cntnt" id="figlgndA982"><h4 id="A982"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A982/?report=objectonly" target="object" rid-ob="figobA982">Figure 6-7</a></h4><p class="float-caption no_bottom_margin">DNA transcription produces a single-stranded RNA molecule that is complementary to one strand of DNA. </p></div></div><p>Transcription, however, differs from <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> replication in several crucial ways. Unlike a newly formed DNA strand, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> strand does not remain hydrogen-bonded to the DNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> strand. Instead, just behind the region where the ribonucleotides are being added, the RNA chain is displaced and the DNA helix re-forms. Thus, the RNA molecules produced by transcription are released from the DNA template as single strands. In addition, because they are copied from only a limited region of the DNA, RNA molecules are much shorter than DNA molecules. A DNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> in a human <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4977/" role="button" aria-expanded="false" aria-haspopup="true">chromosome</a> can be up to 250 million <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a>-pairs long; in contrast, most RNAs are no more than a few thousand nucleotides long, and many are considerably shorter.</p><p>The enzymes that perform transcription are called <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" class="def" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerases</a>. Like the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5092/" role="button" aria-expanded="false" aria-haspopup="true">DNA polymerase</a> that catalyzes DNA replication (discussed in Chapter 5), <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> polymerases catalyze the formation of the phosphodiester bonds that link the nucleotides together to form a linear chain. The <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> moves stepwise along the DNA, unwinding the DNA helix just ahead of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/" role="button" aria-expanded="false" aria-haspopup="true">active site</a> for polymerization to expose a new region of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> strand for <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/" role="button" aria-expanded="false" aria-haspopup="true">complementary</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pairing. In this way, the growing RNA chain is extended by one <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> at a time in the 5′-to-3′ direction (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A983/?report=objectonly" target="object" rid-figpopup="figA983" rid-ob="figobA983">Figure 6-8</a>). The substrates are <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5565/" role="button" aria-expanded="false" aria-haspopup="true">nucleoside</a> triphosphates (ATP, CTP, UTP, and GTP); as for DNA replication, a hydrolysis of high-energy bonds provides the energy needed to drive the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> forward (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A754/figure/A758/?report=objectonly" target="object">Figure 5-4</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA983" co-legend-rid="figlgndA983"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A983/?report=objectonly" target="object" title="Figure 6-8" class="img_link icnblk_img figpopup" rid-figpopup="figA983" rid-ob="figobA983"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f8.gif" src-large="/books/NBK26887/bin/ch6f8.jpg" alt="Figure 6-8. DNA is transcribed by the enzyme RNA polymerase."></a><div class="icnblk_cntnt" id="figlgndA983"><h4 id="A983"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A983/?report=objectonly" target="object" rid-ob="figobA983">Figure 6-8</a></h4><p class="float-caption no_bottom_margin">DNA is transcribed by the enzyme RNA polymerase. The RNA polymerase <i>(pale blue)</i> moves stepwise along the DNA, unwinding the DNA helix at its active site. As it progresses, the polymerase adds nucleotides (here, <i>small “T” shapes)</i> one by <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A983/?report=objectonly" target="object" rid-ob="figobA983">(more...)</a></p></div></div><p>The almost immediate release of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> strand from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> as it is synthesized means that many RNA copies can be made from the same <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> in a relatively short time, the synthesis of additional RNA molecules being started before the first RNA is completed (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A984/?report=objectonly" target="object" rid-figpopup="figA984" rid-ob="figobA984">Figure 6-9</a>). When <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> molecules follow hard on each other's heels in this way, each moving at about 20 nucleotides per second (the speed in eucaryotes), over a thousand transcripts can be synthesized in an hour from a single gene.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA984" co-legend-rid="figlgndA984"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A984/?report=objectonly" target="object" title="Figure 6-9" class="img_link icnblk_img figpopup" rid-figpopup="figA984" rid-ob="figobA984"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f9.gif" src-large="/books/NBK26887/bin/ch6f9.jpg" alt="Figure 6-9. Transcription of two genes as observed under the electron microscope."></a><div class="icnblk_cntnt" id="figlgndA984"><h4 id="A984"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A984/?report=objectonly" target="object" rid-ob="figobA984">Figure 6-9</a></h4><p class="float-caption no_bottom_margin">Transcription of two genes as observed under the electron microscope. The micrograph shows many molecules of RNA polymerase simultaneously transcribing each of two adjacent genes. Molecules of RNA polymerase are visible as a series of dots along the DNA <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A984/?report=objectonly" target="object" rid-ob="figobA984">(more...)</a></p></div></div><p>Although <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> catalyzes essentially the same chemical <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> as <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5092/" role="button" aria-expanded="false" aria-haspopup="true">DNA polymerase</a>, there are some important differences between the two enzymes. First, and most obvious, RNA polymerase catalyzes the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5397/" role="button" aria-expanded="false" aria-haspopup="true">linkage</a> of ribonucleotides, not deoxyribonucleotides. Second, unlike the DNA polymerases involved in DNA replication, RNA polymerases can start an RNA chain without a primer. This difference may exist because transcription need not be as accurate as DNA replication (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A754/table/A767/?report=objectonly" target="object">Table 5-1</a>, p. 243). Unlike DNA, RNA does not permanently store genetic information in cells. RNA polymerases make about one mistake for every 10<sup>4</sup> nucleotides copied into RNA (compared with an error rate for direct copying by DNA polymerase of about one in 10<sup>7</sup> nucleotides), and the consequences of an error in RNA transcription are much less significant than that in DNA replication.</p><p>Although <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> polymerases are not nearly as accurate as the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> polymerases that replicate DNA, they nonetheless have a modest proofreading mechanism. If the incorrect ribonucleotide is added to the growing RNA chain, the polymerase can back up, and the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/" role="button" aria-expanded="false" aria-haspopup="true">active site</a> of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> can perform an excision <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> that mimics the reverse of the polymerization reaction, except that water instead of pyrophosphate is used (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A754/figure/A758/?report=objectonly" target="object">Figure 5-4</a>). <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> hovers around a misincorporated ribonucleotide longer than it does for a correct addition, causing excision to be favored for incorrect nucleotides. However, RNA polymerase also excises many correct bases as part of the cost for improved accuracy.</p></div><div id="A985"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A985_" class="ui-helper-clearfix">Cells Produce Several Types of RNA</h2><p>The majority of genes carried in a cell's <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> specify the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/" role="button" aria-expanded="false" aria-haspopup="true">amino acid</a> sequence of proteins; the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules that are copied from these genes (which ultimately direct the synthesis of proteins) are called <b><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5448/" role="button" aria-expanded="false" aria-haspopup="true">messenger RNA (mRNA)</a></b> molecules. The final product of a minority of genes, however, is the RNA itself. Careful analysis of the complete DNA sequence of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a> of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5937/" role="button" aria-expanded="false" aria-haspopup="true">yeast</a> <i>S. cerevisiae</i> has uncovered well over 750 genes (somewhat more than 10% of the total number of yeast genes) that produce RNA as their final product, although this number includes multiple copies of some highly repeated genes. These RNAs, like proteins, serve as enzymatic and structural components for a wide variety of processes in the cell. In Chapter 5 we encountered one of those RNAs, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> carried by the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5858/" role="button" aria-expanded="false" aria-haspopup="true">telomerase</a>. Although not all of their functions are known, we see in this chapter that some <i>small nuclear RNA</i> (<i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5812/" role="button" aria-expanded="false" aria-haspopup="true">snRNA</a></i>) molecules direct the splicing of pre-mRNA to form mRNA, that <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5753/" role="button" aria-expanded="false" aria-haspopup="true">ribosomal RNA (rRNA)</a></i> molecules form the core of ribosomes, and that <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5886/" role="button" aria-expanded="false" aria-haspopup="true">transfer RNA (tRNA)</a></i> molecules form the adaptors that select amino acids and hold them in place on a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/" role="button" aria-expanded="false" aria-haspopup="true">ribosome</a> for incorporation into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A986/?report=objectonly" target="object" rid-figpopup="figA986" rid-ob="figobA986">Table 6-1</a>).</p><div class="iconblock whole_rhythm clearfix ten_col table-wrap" id="figA986"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A986/?report=objectonly" target="object" title="Table 6-1" class="img_link icnblk_img figpopup" rid-figpopup="figA986" rid-ob="figobA986"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/saved_resource" src-large="/books/NBK26887/table/A986/?report=previmg" alt="Table 6-1. Principal Types of RNAs Produced in Cells."></a><div class="icnblk_cntnt"><h4 id="A986"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A986/?report=objectonly" target="object" rid-ob="figobA986">Table 6-1</a></h4><p class="float-caption no_bottom_margin">Principal Types of RNAs Produced in Cells. </p></div></div><p>Each transcribed segment of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> is called a <i>transcription unit</i>. In eucaryotes, a transcription unit typically carries the information of just one <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>, and therefore codes for either a single <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> or a single <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> (or group of related proteins if the initial RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a> is spliced in more than one way to produce different mRNAs). In bacteria, a set of adjacent genes is often trans-cribed as a unit; the resulting <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> molecule therefore carries the information for several distinct proteins.</p><p>Overall, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> makes up a few percent of a cell's dry weight. Most of the RNA in cells is <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a>; <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> comprises only 3–5% of the total RNA in a typical mammalian cell. The mRNA population is made up of tens of thousands of different species, and there are on average only 10–15 molecules of each species of mRNA present in each cell.</p></div><div id="A987"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A987_" class="ui-helper-clearfix">Signals Encoded in DNA Tell RNA Polymerase Where to Start and Stop</h2><p>To transcribe a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> accurately, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> must recognize where on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a> to start and where to finish. The way in which RNA polymerases perform these tasks differs somewhat between bacteria and eucaryotes. Because the process in bacteria is simpler, we look there first.</p><p>The initiation of transcription is an especially important step in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5163/" role="button" aria-expanded="false" aria-haspopup="true">expression</a> because it is the main point at which the cell regulates which proteins are to be produced and at what rate. Bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> is a multisubunit <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a>. A detachable <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/" role="button" aria-expanded="false" aria-haspopup="true">subunit</a>, called <i>sigma (σ) factor</i>, is largely responsible for its ability to read the signals in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> that tell it where to begin transcribing (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>). RNA polymerase molecules adhere only weakly to the bacterial DNA when they collide with it, and a polymerase <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> typically slides rapidly along the long DNA molecule until it dissociates again. However, when the polymerase slides into a region on the DNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/" role="button" aria-expanded="false" aria-haspopup="true">double helix</a> called a <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" class="def" role="button" aria-expanded="false" aria-haspopup="true">promoter</a>, a special sequence of nucleotides indicating the starting point for RNA synthesis, it binds tightly to it. The polymerase, using its σ factor, recognizes this DNA sequence by making specific contacts with the portions of the bases that are exposed on the outside of the helix <i>(Step 1</i> in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA988" co-legend-rid="figlgndA988"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" title="Figure 6-10" class="img_link icnblk_img figpopup" rid-figpopup="figA988" rid-ob="figobA988"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f10.gif" src-large="/books/NBK26887/bin/ch6f10.jpg" alt="Figure 6-10. The transcription cycle of bacterial RNA polymerase."></a><div class="icnblk_cntnt" id="figlgndA988"><h4 id="A988"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-ob="figobA988">Figure 6-10</a></h4><p class="float-caption no_bottom_margin">The transcription cycle of bacterial RNA polymerase. In step 1, the RNA polymerase holoenzyme (core polymerase plus σ factor) forms and then locates a promoter (see Figure 6-12). The polymerase unwinds the DNA at the position at which transcription <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-ob="figobA988">(more...)</a></p></div></div><p>After the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> binds tightly to the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> in this way, it opens up the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/" role="button" aria-expanded="false" aria-haspopup="true">double helix</a> to expose a short stretch of nucleotides on each strand <i>(Step 2</i> in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>). Unlike a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5087/" role="button" aria-expanded="false" aria-haspopup="true">DNA helicase</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A754/figure/A775/?report=objectonly" target="object">Figure 5-15</a>), this limited opening of the helix does not require the energy of ATP hydrolysis. Instead, the polymerase and DNA both undergo reversible structural changes that result in a more energetically favorable state. With the DNA unwound, one of the two exposed DNA strands acts as a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> for <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/" role="button" aria-expanded="false" aria-haspopup="true">complementary</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pairing with incoming ribonucleotides (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A982/?report=objectonly" target="object" rid-figpopup="figA982" rid-ob="figobA982">Figure 6-7</a>), two of which are joined together by the polymerase to begin an RNA chain. After the first ten or so nucleotides of RNA have been synthesized (a relatively inefficient process during which polymerase synthesizes and discards short <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> oligomers), the σ factor relaxes its tight hold on the polymerase and evenutally dissociates from it. During this process, the polymerase undergoes additional structural changes that enable it to move forward rapidly, transcribing without the σ factor <i>(Step 4</i> in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>). Chain elongation continues (at a speed of approximately 50 nucleotides/sec for bacterial RNA polymerases) until the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> encounters a second signal in the DNA, the <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5863/" class="def" role="button" aria-expanded="false" aria-haspopup="true">terminator</a> (described below), where the polymerase halts and releases both the DNA template and the newly made RNA chain <i>(Step 7</i> in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>). After the polymerase has been released at a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5863/" role="button" aria-expanded="false" aria-haspopup="true">terminator</a>, it reassociates with a free σ factor and searches for a new promoter, where it can begin the process of transcription again.</p><p>Several structural features of bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> make it particularly adept at performing the transcription cycle just described. Once the σ factor positions the polymerase on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a> and the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> has been unwound and pushed to the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/" role="button" aria-expanded="false" aria-haspopup="true">active site</a>, a pair of moveable jaws is thought to clamp onto the DNA (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A989/?report=objectonly" target="object" rid-figpopup="figA989" rid-ob="figobA989">Figure 6-11</a>). When the first 10 nucleotides have been transcribed, the dissociation of σ allows a flap at the back of the polymerase to close to form an exit tunnel through which the newly made RNA leaves the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a>. With the polymerase now functioning in its elongation mode, a rudder-like structure in the enzyme continuously pries apart the DNA-RNA hybrid formed. We can view the series of conformational changes that takes place during transcription initiation as a successive tightening of the enzyme around the DNA and RNA to ensure that it does not dissociate before it has finished transcribing a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>. If an RNA polymerase does dissociate prematurely, it cannot resume synthesis but must start over again at the promoter.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA989" co-legend-rid="figlgndA989"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A989/?report=objectonly" target="object" title="Figure 6-11" class="img_link icnblk_img figpopup" rid-figpopup="figA989" rid-ob="figobA989"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f11.gif" src-large="/books/NBK26887/bin/ch6f11.jpg" alt="Figure 6-11. The structure of a bacterial RNA polymerase."></a><div class="icnblk_cntnt" id="figlgndA989"><h4 id="A989"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A989/?report=objectonly" target="object" rid-ob="figobA989">Figure 6-11</a></h4><p class="float-caption no_bottom_margin">The structure of a bacterial RNA polymerase. Two depictions of the three-dimensional structure of a bacterial RNA polymerase, with the DNA and RNA modeled in. This RNA polymerase is formed from four different subunits, indicated by different colors <i>(right).</i> <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A989/?report=objectonly" target="object" rid-ob="figobA989">(more...)</a></p></div></div><p>How do the signals in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> (termination signals) stop the elongating polymerase? For most bacterial genes a termination signal consists of a string of A-T <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> pairs preceded by a two-fold symmetric DNA sequence, which, when transcribed into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>, folds into a “hairpin” structure through Watson-Crick <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pairing (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>). As the polymerase transcribes across a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5863/" role="button" aria-expanded="false" aria-haspopup="true">terminator</a>, the hairpin may help to wedge open the movable flap on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> and release the RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a> from the exit tunnel. At the same time, the DNA-RNA hybrid in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/" role="button" aria-expanded="false" aria-haspopup="true">active site</a>, which is held together predominantly by U-A base pairs (which are less stable than <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/" role="button" aria-expanded="false" aria-haspopup="true">G</a>-C base pairs because they form two rather than three hydrogen bonds per <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4876/" role="button" aria-expanded="false" aria-haspopup="true">base pair</a>), is not sufficiently strong enough to hold the RNA in place, and it dissociates causing the release of the polymerase from the DNA, perhaps by forcing open its jaws. Thus, in some respects, transcription termination seems to involve a reversal of the structural transitions that happen during initiation. The process of termination also is an example of a common theme in this chapter: the ability of RNA to fold into specific structures figures prominantly in many aspects of decoding the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a>.</p></div><div id="A990"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A990_" class="ui-helper-clearfix">Transcription Start and Stop Signals Are Heterogeneous in Nucleotide Sequence</h2><p>As we have just seen, the processes of transcription initiation and termination involve a complicated series of structural transitions in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a>, and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules. It is perhaps not surprising that the signals encoded in DNA that specify these transitions are difficult for researchers to recognize. Indeed, a comparison of many different bacterial promoters reveals that they are heterogeneous in DNA sequence. Nevertheless, they all contain related sequences, reflecting in part aspects of the DNA that are recognized directly by the σ factor. These common features are often summarized in the form of a <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5023/" role="button" aria-expanded="false" aria-haspopup="true">consensus sequence</a></i> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" target="object" rid-figpopup="figA991" rid-ob="figobA991">Figure 6-12</a>). In general, a consensus <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence is derived by comparing many sequences with the same <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4877/" role="button" aria-expanded="false" aria-haspopup="true">basic</a> function and tallying up the most common nucleotide found at each position. It therefore serves as a summary or “average” of a large number of individual nucleotide sequences.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA991" co-legend-rid="figlgndA991"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" target="object" title="Figure 6-12" class="img_link icnblk_img figpopup" rid-figpopup="figA991" rid-ob="figobA991"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f12.gif" src-large="/books/NBK26887/bin/ch6f12.jpg" alt="Figure 6-12. Consensus sequence for the major class of E. coli promoters.."></a><div class="icnblk_cntnt" id="figlgndA991"><h4 id="A991"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" target="object" rid-ob="figobA991">Figure 6-12</a></h4><p class="float-caption no_bottom_margin">Consensus sequence for the major class of <i>E. coli</i>
promoters.. (A) The promoters are characterized by two hexameric DNA sequences, the -35 sequence and the -10 sequence named for their approximate location relative to the start point of transcription (designated <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" target="object" rid-ob="figobA991">(more...)</a></p></div></div><p>One reason that individual bacterial promoters differ in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> sequence is that the precise sequence determines the strength (or number of initiation events per unit time) of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a>. Evolutionary processes have thus fine-tuned each promoter to initiate as often as necessary and have created a wide spectrum of promoters. Promoters for genes that code for abundant proteins are much stronger than those associated with genes that encode rare proteins, and their <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequences are responsible for these differences.</p><p>Like bacterial promoters, transcription terminators also include a wide range of sequences, with the potential to form a simple <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> structure being the most important common feature. Since an almost unlimited number of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequences have this potential, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5863/" role="button" aria-expanded="false" aria-haspopup="true">terminator</a> sequences are much more heterogeneous than those of promoters.</p><p>We have discussed bacterial promoters and terminators in some detail to illustrate an important point regarding the analysis of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a> sequences. Although we know a great deal about bacterial promoters and terminators and can develop consensus sequences that summarize their most salient features, their variation in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence makes it difficult for researchers (even when aided by powerful computers) to definitively locate them simply by inspection of the nucleotide sequence of a genome. When we encounter analogous types of sequences in eucaryotes, the problem of locating them is even more difficult. Often, additional information, some of it from direct experimentation, is needed to accurately locate the short <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> signals contained in genomes.</p><p>Promoter sequences are asymmetric (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" target="object" rid-figpopup="figA991" rid-ob="figobA991">Figure 6-12</a>), and this feature has important consequences for their arrangement in genomes. Since <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> is double-stranded, two different <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules could in principle be transcribed from any <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>, using each of the two DNA strands as a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a>. However a gene typically has only a single <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a>, and because the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequences of bacterial (as well as eucaryotic) promoters are asymmetric the polymerase can bind in only one orientation. The polymerase thus has no option but to transcribe the one DNA strand, since it can synthesize RNA only in the 5′ to 3′ direction (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A992/?report=objectonly" target="object" rid-figpopup="figA992" rid-ob="figobA992">Figure 6-13</a>). The choice of template strand for each gene is therefore <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/" role="button" aria-expanded="false" aria-haspopup="true">determined</a> by the location and orientation of the promoter. Genome sequences reveal that the DNA strand used as the template for RNA synthesis varies from gene to gene (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A993/?report=objectonly" target="object" rid-figpopup="figA993" rid-ob="figobA993">Figure 6-14</a>; see also <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A68/figure/A70/?report=objectonly" target="object">Figure 1-31</a>).


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA992" co-legend-rid="figlgndA992"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A992/?report=objectonly" target="object" title="Figure 6-13" class="img_link icnblk_img figpopup" rid-figpopup="figA992" rid-ob="figobA992"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f13.gif" src-large="/books/NBK26887/bin/ch6f13.jpg" alt="Figure 6-13. The importance of RNA polymerase orientation."></a><div class="icnblk_cntnt" id="figlgndA992"><h4 id="A992"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A992/?report=objectonly" target="object" rid-ob="figobA992">Figure 6-13</a></h4><p class="float-caption no_bottom_margin">The importance of RNA polymerase orientation. The DNA strand serving as template must be traversed in a 3′ to 5′ direction, as illustrated in Figure 6-9. Thus, the direction of RNA polymerase movement determines which of the two DNA strands <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A992/?report=objectonly" target="object" rid-ob="figobA992">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA993" co-legend-rid="figlgndA993"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A993/?report=objectonly" target="object" title="Figure 6-14" class="img_link icnblk_img figpopup" rid-figpopup="figA993" rid-ob="figobA993"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f14.gif" src-large="/books/NBK26887/bin/ch6f14.jpg" alt="Figure 6-14. Directions of transcription along a short portion of a bacterial chromosome."></a><div class="icnblk_cntnt" id="figlgndA993"><h4 id="A993"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A993/?report=objectonly" target="object" rid-ob="figobA993">Figure 6-14</a></h4><p class="float-caption no_bottom_margin">Directions of transcription along a short portion of a bacterial chromosome. Some genes are transcribed using one DNA strand as a template, while others are transcribed using the other DNA strand. The direction of transcription is determined by the promoter <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A993/?report=objectonly" target="object" rid-ob="figobA993">(more...)</a></p></div></div><p>Having considered transcription in bacteria, we now turn to the situation in eucaryotes, where the synthesis of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules is a much more elaborate affair.</p></div><div id="A994"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A994_" class="ui-helper-clearfix">Transcription Initiation in Eucaryotes Requires Many Proteins</h2><p>In contrast to bacteria, which contain a single type of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a>, eucaryotic nuclei have three, called <i>RNA polymerase I, RNA polymerase II,</i> and <i>RNA polymerase III</i>. The three polymerases are structurally similar to one another (and to the bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a>). They share some common subunits and many structural features, but they transcribe different types of genes (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A995/?report=objectonly" target="object" rid-figpopup="figA995" rid-ob="figobA995">Table 6-2</a>). RNA polymerases I and III transcribe the genes encoding transfer RNA, ribosomal RNA, and various small RNAs. RNA polymerase II transcribes the vast majority of genes, including all those that encode proteins, and our subsequent discussion therefore focuses on this enzyme.</p><div class="iconblock whole_rhythm clearfix ten_col table-wrap" id="figA995"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A995/?report=objectonly" target="object" title="Table 6-2" class="img_link icnblk_img figpopup" rid-figpopup="figA995" rid-ob="figobA995"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/saved_resource(1)" src-large="/books/NBK26887/table/A995/?report=previmg" alt="Table 6-2. The Three RNA Polymerases in Eucaryotic Cells."></a><div class="icnblk_cntnt"><h4 id="A995"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A995/?report=objectonly" target="object" rid-ob="figobA995">Table 6-2</a></h4><p class="float-caption no_bottom_margin">The Three RNA Polymerases in Eucaryotic Cells. </p></div></div><p>Although eucaryotic <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II has many structural similarities to bacterial RNA polymerase (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A996/?report=objectonly" target="object" rid-figpopup="figA996" rid-ob="figobA996">Figure 6-15</a>), there are several important differences in the way in which the bacterial and eucaryotic enzymes function, two of which concern us immediately.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA996" co-legend-rid="figlgndA996"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A996/?report=objectonly" target="object" title="Figure 6-15" class="img_link icnblk_img figpopup" rid-figpopup="figA996" rid-ob="figobA996"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f15.gif" src-large="/books/NBK26887/bin/ch6f15.jpg" alt="Figure 6-15. Structural similarity between a bacterial RNA polymerase and a eucaryotic RNA polymerase II."></a><div class="icnblk_cntnt" id="figlgndA996"><h4 id="A996"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A996/?report=objectonly" target="object" rid-ob="figobA996">Figure 6-15</a></h4><p class="float-caption no_bottom_margin">Structural similarity between a bacterial RNA polymerase and a eucaryotic RNA polymerase II. Regions of the two RNA polymerases that have similar structures are indicated in <i>green</i>. The eucaryotic polymerase is larger than the bacterial enzyme (12 subunits <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A996/?report=objectonly" target="object" rid-ob="figobA996">(more...)</a></p></div></div>
<dl class="temp-labeled-list"><dt>1.</dt><dd id="A997"><p class="no_top_margin">While bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> (with σ factor as one of its subunits) is able to initiate transcription on a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5327/" role="button" aria-expanded="false" aria-haspopup="true">in vitro</a></i> without the help of additional proteins, eucaryotic RNA polymerases cannot. They require the help of a large set of proteins called <i>general transcription factors</i>, which must assemble at the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a> with the polymerase before the polymerase can begin transcription.</p></dd><dt>2.</dt><dd id="A998"><p class="no_top_margin">Eucaryotic transcription initiation must deal with the packing of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> into nucleosomes and higher order forms of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4974/" role="button" aria-expanded="false" aria-haspopup="true">chromatin</a> structure, features absent from bacterial chromosomes.</p></dd></dl>
</div><div id="A999"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A999_" class="ui-helper-clearfix">RNA Polymerase II Requires General Transcription Factors</h2><p>The discovery that, unlike bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a>, purified eucaryotic RNA polymerase II could not initiate transcription <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5327/" role="button" aria-expanded="false" aria-haspopup="true">in vitro</a></i> led to the discovery and purification of the additional factors required for this process. These <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5217/" class="def" role="button" aria-expanded="false" aria-haspopup="true">general transcription factors</a> help to position the RNA polymerase correctly at the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a>, aid in pulling apart the two strands of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> to allow transcription to begin, and release RNA polymerase from the promoter into the elongation mode once transcription has begun. The proteins are “general” because they assemble on all promoters used by RNA polymerase II; consisting of a set of interacting proteins, they are designated as <i>TFII</i> (for <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5882/" role="button" aria-expanded="false" aria-haspopup="true">transcription factor</a> for polymerase II), and listed as TFIIA, TFIIB, and so on. In a broad sense, the eucaryotic general transcription factors carry out functions equivalent to those of the σ factor in bacteria.</p><p>
<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-figpopup="figA1000" rid-ob="figobA1000">Figure 6-16</a> shows how the general transcription factors assemble <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5327/" role="button" aria-expanded="false" aria-haspopup="true">in vitro</a></i> at promoters used by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II. The assembly process starts with the binding of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5217/" role="button" aria-expanded="false" aria-haspopup="true">general transcription factor</a> TFIID to a short double-helical <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> sequence primarily composed of T and A nucleotides. For this reason, this sequence is known as the TATA sequence, or <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5856/" class="def" role="button" aria-expanded="false" aria-haspopup="true">TATA box</a>, and the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/" role="button" aria-expanded="false" aria-haspopup="true">subunit</a> of TFIID that recognizes it is called TBP (for TATA-binding <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>). The <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5856/" role="button" aria-expanded="false" aria-haspopup="true">TATA box</a> is typically located 25 nucleotides upstream from the transcription start site. It is not the only DNA sequence that signals the start of transcription (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1001/?report=objectonly" target="object" rid-figpopup="figA1001" rid-ob="figobA1001">Figure 6-17</a>), but for most polymerase II promoters, it is the most important. The binding of TFIID causes a large distortion in the DNA of the TATA box (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1002/?report=objectonly" target="object" rid-figpopup="figA1002" rid-ob="figobA1002">Figure 6-18</a>). This distortion is thought to serve as a physical landmark for the location of an active <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a> in the midst of a very large <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a>, and it brings DNA sequences on both sides of the distortion together to allow for subsequent protein assembly steps. Other factors are then assembled, along with RNA polymerase II, to form a complete <i>transcription initiation <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a></i> (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-figpopup="figA1000" rid-ob="figobA1000">Figure 6-16</a>).



</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1000" co-legend-rid="figlgndA1000"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" title="Figure 6-16" class="img_link icnblk_img figpopup" rid-figpopup="figA1000" rid-ob="figobA1000"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f16.gif" src-large="/books/NBK26887/bin/ch6f16.jpg" alt="Figure 6-16. Initiation of transcription of a eucaryotic gene by RNA polymerase II."></a><div class="icnblk_cntnt" id="figlgndA1000"><h4 id="A1000"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-ob="figobA1000">Figure 6-16</a></h4><p class="float-caption no_bottom_margin">Initiation of transcription of a eucaryotic gene by RNA polymerase II. To begin transcription, RNA polymerase requires a number of general transcription factors (called TFIIA, TFIIB, and so on). (A) The promoter contains a DNA sequence called the TATA <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-ob="figobA1000">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1001" co-legend-rid="figlgndA1001"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1001/?report=objectonly" target="object" title="Figure 6-17" class="img_link icnblk_img figpopup" rid-figpopup="figA1001" rid-ob="figobA1001"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f17.gif" src-large="/books/NBK26887/bin/ch6f17.jpg" alt="Figure 6-17. Consensus sequences found in the vicinity of eucaryotic RNA polymerase II start points."></a><div class="icnblk_cntnt" id="figlgndA1001"><h4 id="A1001"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1001/?report=objectonly" target="object" rid-ob="figobA1001">Figure 6-17</a></h4><p class="float-caption no_bottom_margin">Consensus sequences found in the vicinity of eucaryotic RNA polymerase II start points. The name given to each consensus sequence <i>(first column)</i> and the general transcription factor that recognizes it <i>(last column)</i> are indicated. N indicates any nucleotide, <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1001/?report=objectonly" target="object" rid-ob="figobA1001">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1002" co-legend-rid="figlgndA1002"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1002/?report=objectonly" target="object" title="Figure 6-18" class="img_link icnblk_img figpopup" rid-figpopup="figA1002" rid-ob="figobA1002"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f18.gif" src-large="/books/NBK26887/bin/ch6f18.jpg" alt="Figure 6-18. Three-dimensional structure of TBP (TATA-binding protein) bound to DNA."></a><div class="icnblk_cntnt" id="figlgndA1002"><h4 id="A1002"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1002/?report=objectonly" target="object" rid-ob="figobA1002">Figure 6-18</a></h4><p class="float-caption no_bottom_margin">Three-dimensional structure of TBP (TATA-binding protein) bound to DNA. The TBP is the subunit of the general transcription factor TFIID that is responsible for recognizing and binding to the TATA box sequence in the DNA <i>(red).</i> The unique DNA bending <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1002/?report=objectonly" target="object" rid-ob="figobA1002">(more...)</a></p></div></div><p>After <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II has been guided onto the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> to form a transcription initiation <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a>, it must gain access to the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> strand at the transcription start point. This step is aided by one of the general transcription factors, TFIIH, which contains a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5087/" role="button" aria-expanded="false" aria-haspopup="true">DNA helicase</a>. Next, like the bacterial polymerase, polymerase II remains at the promoter, synthesizing short lengths of RNA until it undergoes a conformational change and is released to begin transcribing a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>. A key step in this release is the addition of phosphate groups to the “tail” of the RNA polymerase (known as the CTD or C-terminal <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/" role="button" aria-expanded="false" aria-haspopup="true">domain</a>). This <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5629/" role="button" aria-expanded="false" aria-haspopup="true">phosphorylation</a> is also catalyzed by TFIIH, which, in addition to a helicase, contains a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5691/" role="button" aria-expanded="false" aria-haspopup="true">protein kinase</a> as one of its subunits (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-figpopup="figA1000" rid-ob="figobA1000">Figure 6-16</a>, <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-figpopup="figA1000" rid-ob="figobA1000">D</a> and <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" target="object" rid-figpopup="figA1000" rid-ob="figobA1000">E</a>). The polymerase can then disengage from the cluster of general transcription factors, undergoing a series of conformational changes that tighten its interaction with DNA and acquiring new proteins that allow it to transcribe for long distances without dissociating.</p><p>Once the polymerase II has begun elongating the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a>, most of the general transcription factors are released from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> so that they are available to initiate another round of transcription with a new <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a>. As we see shortly, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5629/" role="button" aria-expanded="false" aria-haspopup="true">phosphorylation</a> of the tail of RNA polymerase II also causes components of the RNA processing machinery to load onto the polymerase and thus be in position to modify the newly transcribed RNA as it emerges from the polymerase.</p></div><div id="A1003"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1003_" class="ui-helper-clearfix">Polymerase II Also Requires Activator, Mediator, and Chromatin-modifying Proteins</h2><p>The model for transcription initiation just described was established by studying the action of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II and its general transcription factors on purified <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> templates <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5327/" role="button" aria-expanded="false" aria-haspopup="true">in vitro</a></i>. However, as discussed in Chapter 4, DNA in eucaryotic cells is packaged into nucleosomes, which are further arranged in higher-order <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4974/" role="button" aria-expanded="false" aria-haspopup="true">chromatin</a> structures. As a result, transcription initiation in a eucaryotic cell is more <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> and requires more proteins than it does on purified DNA. First, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> regulatory proteins known as <i>transcriptional activators</i> bind to specific sequences in DNA and help to attract RNA polymerase II to the start point of transcription (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1004/?report=objectonly" target="object" rid-figpopup="figA1004" rid-ob="figobA1004">Figure 6-19</a>). This attraction is needed to help the RNA polymerase and the general transcription factors in overcoming the difficulty of binding to DNA that is packaged in chromatin. We discuss the role of activators in Chapter 7, because they represent one of the main ways in which cells regulate <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5163/" role="button" aria-expanded="false" aria-haspopup="true">expression</a> of their genes. Here we simply note that their presence on DNA is required for transcription initiation in a eucaryotic cell. Second, eucaryotic transcription initiation <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5328/" role="button" aria-expanded="false" aria-haspopup="true">in vivo</a></i> requires the presence of a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> complex known as the <i>mediator</i>, which allows the activator proteins to communicate properly with the polymerase II and with the general transcription factors. Finally, transcription initiation in the cell often requires the local recruitment of chromatin-modifying enzymes, including chromatin remodeling complexes and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5283/" role="button" aria-expanded="false" aria-haspopup="true">histone</a> acetylases (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1004/?report=objectonly" target="object" rid-figpopup="figA1004" rid-ob="figobA1004">Figure 6-19</a>). As discussed in Chapter 4, both types of enzymes can allow greater accessibility to the DNA present in chromatin, and by doing so, they facilitate the assembly of the transcription initiation machinery onto DNA.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1004" co-legend-rid="figlgndA1004"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1004/?report=objectonly" target="object" title="Figure 6-19" class="img_link icnblk_img figpopup" rid-figpopup="figA1004" rid-ob="figobA1004"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f19.gif" src-large="/books/NBK26887/bin/ch6f19.jpg" alt="Figure 6-19. Transcription initiation by RNA polymerase II in a eucaryotic cell."></a><div class="icnblk_cntnt" id="figlgndA1004"><h4 id="A1004"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1004/?report=objectonly" target="object" rid-ob="figobA1004">Figure 6-19</a></h4><p class="float-caption no_bottom_margin">Transcription initiation by RNA polymerase II in a eucaryotic cell. Transcription initiation <i>in vivo</i> requires the presence of transcriptional activator proteins. As described in Chapter 7, these proteins bind to specific short sequences in DNA. Although <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1004/?report=objectonly" target="object" rid-ob="figobA1004">(more...)</a></p></div></div><p>As illustrated in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1004/?report=objectonly" target="object" rid-figpopup="figA1004" rid-ob="figobA1004">Figure 6-19</a>, many proteins (well over one hundred individual subunits) must assemble at the start point of transcription to initiate transcription in a eucaryotic cell. The order of assembly of these proteins is probably different for different genes and therefore may not follow a prescribed pathway. In fact, some of these different <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> assemblies may interact with each other away from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> and be brought to DNA as preformed subcomplexes. For example, the mediator, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II, and some of the general transcription factors can bind to each other in the nucleoplasm and be brought to the DNA as a unit. We return to this issue in Chapter 7, where we discuss the many ways eucaryotic cells can regulate the process of transcription initiation.</p></div><div id="A1005"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1005_" class="ui-helper-clearfix">Transcription Elongation Produces Superhelical Tension in DNA</h2><p>Once it has initiated transcription, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> does not proceed smoothly along a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a>; rather it moves jerkily, pausing at some sequences and rapidly transcribing through others. Elongating RNA polymerases, both bacterial and eucaryotic, are associated with a series of <i>elongation factors</i>, proteins that decrease the likelihood that RNA polymerase will dissociate before it reaches the end of a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>. These factors typically associate with RNA polymerase shortly after initiation has occurred and help polymerases to move through the wide variety of different DNA sequences that are found in genes. Eucaryotic RNA polymerases must also contend with <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4974/" role="button" aria-expanded="false" aria-haspopup="true">chromatin</a> structure as they move along a DNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a>. Experiments have shown that bacterial polymerases, which never encounter nucleosomes <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5328/" role="button" aria-expanded="false" aria-haspopup="true">in vivo</a></i>, can nonetheless transcribe through them <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5327/" role="button" aria-expanded="false" aria-haspopup="true">in vitro</a>,</i> suggesting that a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5566/" role="button" aria-expanded="false" aria-haspopup="true">nucleosome</a> is easily traversed. However, eucaryotic polymerases have to move through forms of chromatin that are more compact than a simple nucleosome. It therefore seems likely that they transcribe with the aid of chromatin remodeling complexes (see pp. 212–213). These complexes may move with the polymerase or may simply seek out and rescue the occasional stalled polymerase. In addition, some elongation factors associated with eucaryotic RNA polymerase facilitate transcription through nucleosomes without requiring additional energy. It is not yet understood how this is accomplished, but these proteins may help to dislodge parts of the nucleosome core as the polymerase transcribes the DNA of a nucleosome.</p><p>There is yet another barrier to elongating polymerases, both bacterial and eucaryotic. To discuss this issue, we need first to consider a subtle property inherent in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/" role="button" aria-expanded="false" aria-haspopup="true">double helix</a> called <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5096/" class="def" role="button" aria-expanded="false" aria-haspopup="true">DNA supercoiling</a>. <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5096/" role="button" aria-expanded="false" aria-haspopup="true">DNA supercoiling</a> represents a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/" role="button" aria-expanded="false" aria-haspopup="true">conformation</a> that DNA will adopt in response to superhelical tension; conversely, creating various loops or coils in the helix can create such tension. A simple way of visualizing the topological constraints that cause DNA supercoiling is illustrated in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1006/?report=objectonly" target="object" rid-figpopup="figA1006" rid-ob="figobA1006">Figure 6-20A</a>. There are approximately 10 <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> pairs for every helical turn in a DNA double helix. Imagine a helix whose two ends are fixed with respect to each other (as they are in a DNA circle, such as a bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4977/" role="button" aria-expanded="false" aria-haspopup="true">chromosome</a>, or in a tightly clamped loop, as is thought to exist in eucaryotic chromosomes). In this case, one large DNA supercoil will form to compensate for each 10 nucleotide pairs that are opened (unwound). The formation of this supercoil is energetically favorable because it restores a normal helical twist to the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-paired regions that remain, which would otherwise need to be overwound because of the fixed ends.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1006" co-legend-rid="figlgndA1006"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1006/?report=objectonly" target="object" title="Figure 6-20" class="img_link icnblk_img figpopup" rid-figpopup="figA1006" rid-ob="figobA1006"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f20.gif" src-large="/books/NBK26887/bin/ch6f20.jpg" alt="Figure 6-20. Superhelical tension in DNA causes DNA supercoiling."></a><div class="icnblk_cntnt" id="figlgndA1006"><h4 id="A1006"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1006/?report=objectonly" target="object" rid-ob="figobA1006">Figure 6-20</a></h4><p class="float-caption no_bottom_margin">Superhelical tension in DNA causes DNA supercoiling. (A) For a DNA molecule with one free end (or a nick in one strand that serves as a swivel), the DNA double helix rotates by one turn for every 10 nucleotide pairs opened. (B) If rotation is prevented, <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1006/?report=objectonly" target="object" rid-ob="figobA1006">(more...)</a></p></div></div><p>Superhelical tension is also created as <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> moves along a stretch of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> that is anchored at its ends (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1006/?report=objectonly" target="object" rid-figpopup="figA1006" rid-ob="figobA1006">Figure 6-20C</a>). As long as the polymerase is not free to rotate rapidly (and such rotation is unlikely given the size of RNA polymerases and their attached transcripts), a moving polymerase generates positive superhelical tension in the DNA in front of it and negative helical tension behind it. For eucaryotes, this situation is thought to provide a bonus: the positive superhelical tension ahead of the polymerase makes the DNA helix more difficult to open, but this tension should facilitate the unwrapping of DNA in nucleosomes, as the release of DNA from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5283/" role="button" aria-expanded="false" aria-haspopup="true">histone</a> core helps to relax positive superhelical tension.</p><p>Any <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> that propels itself alone along a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> strand of a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/" role="button" aria-expanded="false" aria-haspopup="true">double helix</a> tends to generate superhelical tension. In eucaryotes, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5097/" role="button" aria-expanded="false" aria-haspopup="true">DNA topoisomerase</a> enzymes rapidly remove this superhelical tension (see p. 251). But, in bacteria, a specialized topoisomerase called <i>DNA gyrase</i> uses the energy of ATP hydrolysis to <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5708/" role="button" aria-expanded="false" aria-haspopup="true">pump</a> supercoils continuously into the DNA, thereby maintaining the DNA under constant tension. These are <i>negative supercoils,</i> having the opposite handedness from the <i>positive supercoils</i> that form when a region of DNA helix opens (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1006/?report=objectonly" target="object" rid-figpopup="figA1006" rid-ob="figobA1006">Figure 6-20B</a>). These negative supercoils are removed from bacterial DNA whenever a region of helix opens, reducing the superhelical tension. DNA gyrase therefore makes the opening of the DNA helix in bacteria energetically favorable compared with helix opening in DNA that is not supercoiled. For this reason, it usually facilitates those genetic processes in bacteria, including the initiation of transcription by bacterial <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a>, that require helix opening (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" target="object" rid-figpopup="figA988" rid-ob="figobA988">Figure 6-10</a>).</p></div><div id="A1007"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1007_" class="ui-helper-clearfix">Transcription Elongation in Eucaryotes Is Tightly Coupled To RNA Processing</h2><p>We have seen that bacterial mRNAs are synthesized solely by the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> starting and stopping at specific spots on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a>. The situation in eucaryotes is substantially different. In particular, transcription is only the first step in a series of reactions that includes the covalent modification of both ends of the RNA and the removal of <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences</i> that are discarded from the middle of the RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a> by the process of <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a></i> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1008/?report=objectonly" target="object" rid-figpopup="figA1008" rid-ob="figobA1008">Figure 6-21</a>). The modifications of the ends of eucaryotic <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> are <i>capping</i> on the 5′ end and <i>polyadenylation</i> of the 3′ end (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1009/?report=objectonly" target="object" rid-figpopup="figA1009" rid-ob="figobA1009">Figure 6-22</a>). These special ends allow the cell to assess whether both ends of an mRNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> are present (and the message is therefore intact) before it exports the RNA sequence from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a> for translation into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>. In Chapter 4, we saw that a typical eucaryotic <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> is present in the genome as short blocks of protein-coding sequence (exons) separated by long introns, and RNA splicing is the critically important step in which the different portions of a protein coding sequence are joined together. As we describe next, RNA splicing also provides higher eucaryotes with the ability to synthesize several different proteins from the same gene.


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1008" co-legend-rid="figlgndA1008"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1008/?report=objectonly" target="object" title="Figure 6-21" class="img_link icnblk_img figpopup" rid-figpopup="figA1008" rid-ob="figobA1008"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f21.gif" src-large="/books/NBK26887/bin/ch6f21.jpg" alt="Figure 6-21. Summary of the steps leading from gene to protein in eucaryotes and bacteria."></a><div class="icnblk_cntnt" id="figlgndA1008"><h4 id="A1008"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1008/?report=objectonly" target="object" rid-ob="figobA1008">Figure 6-21</a></h4><p class="float-caption no_bottom_margin">Summary of the steps leading from gene to protein in eucaryotes and bacteria. The final level of a protein in the cell depends on the efficiency of each step and on the rates of degradation of the RNA and protein molecules. (A) In eucaryotic cells the <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1008/?report=objectonly" target="object" rid-ob="figobA1008">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1009" co-legend-rid="figlgndA1009"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1009/?report=objectonly" target="object" title="Figure 6-22" class="img_link icnblk_img figpopup" rid-figpopup="figA1009" rid-ob="figobA1009"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f22.gif" src-large="/books/NBK26887/bin/ch6f22.jpg" alt="Figure 6-22. A comparison of the structures of procaryotic and eucaryotic mRNA molecules."></a><div class="icnblk_cntnt" id="figlgndA1009"><h4 id="A1009"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1009/?report=objectonly" target="object" rid-ob="figobA1009">Figure 6-22</a></h4><p class="float-caption no_bottom_margin">A comparison of the structures of procaryotic and eucaryotic mRNA molecules. (A) The 5′ and 3′ ends of a bacterial mRNA are the unmodified ends of the chain synthesized by the RNA polymerase, which initiates and terminates transcription <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1009/?report=objectonly" target="object" rid-ob="figobA1009">(more...)</a></p></div></div><p>These <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> processing steps are tightly coupled to transcription elongation by an ingenious mechanism. As discussed previously, a key step of the transition of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II to the elongation mode of RNA synthesis is an extensive <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5629/" role="button" aria-expanded="false" aria-haspopup="true">phosphorylation</a> of the RNA polymerase II tail, called the CTD. This C-terminal <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/" role="button" aria-expanded="false" aria-haspopup="true">domain</a> of the largest <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/" role="button" aria-expanded="false" aria-haspopup="true">subunit</a> consists of a long tandem array of a repeated seven-amino-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4761/" role="button" aria-expanded="false" aria-haspopup="true">acid</a> sequence, containing two serines per repeat that can be phosphorylated. Because there are 52 repeats in the CTD of human RNA polymerase II, its complete phosphorylation would add 104 negatively charged phosphate groups to the polymerase. This phosphorylation step not only dissociates the RNA polymerase II from other proteins present at the start point of transcription, it also allows a new set of proteins to associate with the RNA polymerase tail that function in transcription elongation and pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> processing. As discussed next, some of these processing proteins seem to “hop” from the polymerase tail onto the nascent RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> to begin processing it as it emerges from the RNA polymerase. Thus, RNA polymerase II in its elongation mode can be viewed as an RNA factory that both transcribes <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> into RNA and processes the RNA it produces (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1010/?report=objectonly" target="object" rid-figpopup="figA1010" rid-ob="figobA1010">Figure 6-23</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1010" co-legend-rid="figlgndA1010"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1010/?report=objectonly" target="object" title="Figure 6-23" class="img_link icnblk_img figpopup" rid-figpopup="figA1010" rid-ob="figobA1010"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f23.gif" src-large="/books/NBK26887/bin/ch6f23.jpg" alt="Figure 6-23. The “RNA factory” concept for eucaryotic RNA polymerase II."></a><div class="icnblk_cntnt" id="figlgndA1010"><h4 id="A1010"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1010/?report=objectonly" target="object" rid-ob="figobA1010">Figure 6-23</a></h4><p class="float-caption no_bottom_margin">The “RNA factory” concept for eucaryotic RNA polymerase II. Not only does the polymerase transcribe DNA into RNA, but it also carries pre-mRNA-processing proteins on its tail, which are then transferred to the nascent RNA at the appropriate <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1010/?report=objectonly" target="object" rid-ob="figobA1010">(more...)</a></p></div></div></div><div id="A1011"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1011_" class="ui-helper-clearfix">RNA Capping Is the First Modification of Eucaryotic Pre-mRNAs</h2><p>As soon as <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II has produced about 25 nucleotides of RNA, the 5′ end of the new RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> is modified by addition of a “cap” that consists of a modified guanine <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1009/?report=objectonly" target="object" rid-figpopup="figA1009" rid-ob="figobA1009">Figure 6-22B</a>). The capping <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> is performed by three enzymes acting in succession: one (a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5619/" role="button" aria-expanded="false" aria-haspopup="true">phosphatase</a>) removes one phosphate from the 5′ end of the nascent RNA, another (a guanyl transferase) adds a GMP in a reverse <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5397/" role="button" aria-expanded="false" aria-haspopup="true">linkage</a> (5′ to 5′ instead of 5′ to 3′), and a third (a methyl transferase) adds a methyl group to the guanosine (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1012/?report=objectonly" target="object" rid-figpopup="figA1012" rid-ob="figobA1012">Figure 6-24</a>). Because all three enzymes bind to the phosphorylated RNA polymerase tail, they are poised to modify the 5′ end of the nascent <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a> as soon as it emerges from the polymerase.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1012" co-legend-rid="figlgndA1012"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1012/?report=objectonly" target="object" title="Figure 6-24" class="img_link icnblk_img figpopup" rid-figpopup="figA1012" rid-ob="figobA1012"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f24.gif" src-large="/books/NBK26887/bin/ch6f24.jpg" alt="Figure 6-24. The reactions that cap the 5′ end of each RNA molecule synthesized by RNA polymerase II."></a><div class="icnblk_cntnt" id="figlgndA1012"><h4 id="A1012"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1012/?report=objectonly" target="object" rid-ob="figobA1012">Figure 6-24</a></h4><p class="float-caption no_bottom_margin">The reactions that cap the 5′ end of each RNA molecule synthesized by RNA polymerase II.  The final cap contains a novel 5′-to-5′ linkage between the positively charged 7-methyl G residue and the 5′ end of the RNA transcript <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1012/?report=objectonly" target="object" rid-ob="figobA1012">(more...)</a></p></div></div><p>The 5′-methyl cap signals the 5′ end of eucaryotic mRNAs, and this landmark helps the cell to distinguish mRNAs from the other types of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules present in the cell. For example, RNA polymerases I and III produce uncapped RNAs during transcription, in part because these polymerases lack tails. In the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>, the cap binds a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> called CBC (cap-binding complex), which, as we discuss in subsequent sections, helps the RNA to be properly processed and exported. The 5′ methyl cap also has an important role in the translation of mRNAs in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5055/" role="button" aria-expanded="false" aria-haspopup="true">cytosol</a> as we discuss later in the chapter.</p></div><div id="A1013"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1013_" class="ui-helper-clearfix">RNA Splicing Removes Intron Sequences from Newly Transcribed Pre-mRNAs</h2><p>As discussed in Chapter 4, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> coding sequences of eucaryotic genes are typically interrupted by noncoding intervening sequences (introns). Discovered in 1977, this feature of eucaryotic genes came as a surprise to scientists, who had been, until that time, familiar only with bacterial genes, which typically consist of a continuous stretch of coding <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> that is directly transcribed into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a>. In marked contrast, eucaryotic genes were found to be broken up into small pieces of coding sequence (<i>expressed sequences</i> or <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" class="def" role="button" aria-expanded="false" aria-haspopup="true">exons</a>) interspersed with much longer <i>intervening sequences</i> or <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" class="def" role="button" aria-expanded="false" aria-haspopup="true">introns</a>; thus the coding portion of a eucaryotic <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> is often only a small fraction of the length of the gene (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1014/?report=objectonly" target="object" rid-figpopup="figA1014" rid-ob="figobA1014">Figure 6-25</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1014" co-legend-rid="figlgndA1014"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1014/?report=objectonly" target="object" title="Figure 6-25" class="img_link icnblk_img figpopup" rid-figpopup="figA1014" rid-ob="figobA1014"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f25.gif" src-large="/books/NBK26887/bin/ch6f25.jpg" alt="Figure 6-25. Structure of two human genes showing the arrangement of exons and introns."></a><div class="icnblk_cntnt" id="figlgndA1014"><h4 id="A1014"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1014/?report=objectonly" target="object" rid-ob="figobA1014">Figure 6-25</a></h4><p class="float-caption no_bottom_margin">Structure of two human genes showing the arrangement of exons and introns. (A) The relatively small β-globin gene, which encodes one of the subunits of the oxygen-carrying protein hemoglobin, contains 3 exons (see also Figure 4-7). (B) The much <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1014/?report=objectonly" target="object" rid-ob="figobA1014">(more...)</a></p></div></div><p>Both <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> sequences are transcribed into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>. The intron sequences are removed from the newly synthesized RNA through the process of <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" class="def" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a>. The vast majority of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> that takes place in cells functions in the production of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a>, and our discussion of splicing focuses on this type. It is termed precursor-mRNA (or pre-mRNA) splicing to denote that it occurs on RNA molecules destined to become mRNAs. Only after 5′ and 3′ end processing and splicing have taken place is such RNA termed mRNA.</p><p>Each splicing event removes one <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a>, proceeding through two sequential phosphoryl-transfer reactions known as transesterifications; these join two exons while removing the intron as a “lariat” (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1015/?report=objectonly" target="object" rid-figpopup="figA1015" rid-ob="figobA1015">Figure 6-26</a>). Since the number of phosphate bonds remains the same, these reactions could in principle take place without <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5565/" role="button" aria-expanded="false" aria-haspopup="true">nucleoside</a> triphosphate hydrolysis. However, the machinery that catalyzes pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing is <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a>, consisting of 5 additional <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules and over 50 proteins, and it hydrolyzes many ATP molecules per splicing event. This complexity is presumably needed to ensure that splicing is highly accurate, while also being sufficiently flexible to deal with the enormous variety of introns found in a typical eucaryotic cell. Frequent mistakes in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> would severely harm the cell, as they would result in malfunctioning proteins. We see in Chapter 7 that when rare splicing mistakes do occur, the cell has a “fail-safe” device to eliminate the incorrectly spliced mRNAs.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1015" co-legend-rid="figlgndA1015"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1015/?report=objectonly" target="object" title="Figure 6-26" class="img_link icnblk_img figpopup" rid-figpopup="figA1015" rid-ob="figobA1015"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f26.gif" src-large="/books/NBK26887/bin/ch6f26.jpg" alt="Figure 6-26. The RNA splicing reaction."></a><div class="icnblk_cntnt" id="figlgndA1015"><h4 id="A1015"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1015/?report=objectonly" target="object" rid-ob="figobA1015">Figure 6-26</a></h4><p class="float-caption no_bottom_margin">The RNA splicing reaction. (A) In the first step, a specific adenine nucleotide in the intron sequence (indicated in <i>red)</i> attacks the 5′ splice site and cuts the sugar-phosphate backbone of the RNA at this point. The cut 5′ end of the <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1015/?report=objectonly" target="object" rid-ob="figobA1015">(more...)</a></p></div></div><p>It may seem wasteful to remove large numbers of introns by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a>. In attempting to explain why it occurs, scientists have pointed out that the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a>-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> arrangement would seem to facilitate the emergence of new and useful proteins. Thus, the presence of numerous introns in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> allows <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5220/" role="button" aria-expanded="false" aria-haspopup="true">genetic recombination</a> to readily combine the exons of different genes (see p. 462), allowing genes for new proteins to evolve more easily by the combination of parts of preexisting genes. This idea is supported by the observation, described in Chapter 3, that many proteins in present-day cells resemble patchworks composed from a common set of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> pieces, called protein <i>domains</i>.</p><p><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> also has a present-day advantage. The transcripts of many eucaryotic genes (estimated at 60% of genes in humans) are spliced in a variety of different ways to produce a set of different mRNAs, thereby allowing a corresponding set of different proteins to be produced from the same <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1016/?report=objectonly" target="object" rid-figpopup="figA1016" rid-ob="figobA1016">Figure 6-27</a>). We discuss additional examples of alternative splicing in Chapter 7, as this is also one of the mechanisms that cells use to change <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5163/" role="button" aria-expanded="false" aria-haspopup="true">expression</a> of their genes. Rather than being the wasteful process it may have seemed at first sight, RNA splicing enables eucaryotes to increase the already enormous coding potential of their genomes. We shall return to this idea several times in this chapter and the next, but we first need to describe the cellular machinery that performs this remarkable task.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1016" co-legend-rid="figlgndA1016"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1016/?report=objectonly" target="object" title="Figure 6-27" class="img_link icnblk_img figpopup" rid-figpopup="figA1016" rid-ob="figobA1016"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f27.gif" src-large="/books/NBK26887/bin/ch6f27.jpg" alt="Figure 6-27. Alternative splicing of the α-tropomyosin gene from rat."></a><div class="icnblk_cntnt" id="figlgndA1016"><h4 id="A1016"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1016/?report=objectonly" target="object" rid-ob="figobA1016">Figure 6-27</a></h4><p class="float-caption no_bottom_margin">Alternative splicing of the α-tropomyosin gene from rat.  α-tropomyosin is a coiled-coil protein (see Figure 3-11) that regulates contraction in muscle cells. The primary transcript can be spliced in different ways, as indicated in the <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1016/?report=objectonly" target="object" rid-ob="figobA1016">(more...)</a></p></div></div></div><div id="A1017"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1017_" class="ui-helper-clearfix">Nucleotide Sequences Signal Where Splicing Occurs</h2><p>Introns range in size from about 10 nucleotides to over 100,000 nucleotides. Picking out the precise borders of an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> is very difficult for scientists to do (even with the aid of computers) when confronted by a complete <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a> sequence of a eucaryote. The possibility of alternative splicing compounds the problem of predicting <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> sequences solely from a genome sequence. This difficulty constitutes one of the main barriers to identifying all of the genes in a complete genome sequence, and it is the primary reason that we know only the approximate number of genes in, for example, the human genome. Yet each cell in our body recognizes and rapidly excises the appropriate intron sequences with high fidelity. We have seen that intron sequence removal involves three positions on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>: the 5′ splice site, the 3′ splice site, and the branch point in the intron sequence that forms the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a> of the excised lariat. In pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing, each of these three sites has a consensus <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence that is similar from intron to intron, providing the cell with cues on where splicing is to take place (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1018/?report=objectonly" target="object" rid-figpopup="figA1018" rid-ob="figobA1018">Figure 6-28</a>). However, there is enough variation in each sequence to make it very difficult for scientists to pick out all of the many splicing signals in a genome sequence.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1018" co-legend-rid="figlgndA1018"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1018/?report=objectonly" target="object" title="Figure 6-28" class="img_link icnblk_img figpopup" rid-figpopup="figA1018" rid-ob="figobA1018"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f28.gif" src-large="/books/NBK26887/bin/ch6f28.jpg" alt="Figure 6-28. The consensus nucleotide sequences in an RNA molecule that signal the beginning and the end of most introns in humans."></a><div class="icnblk_cntnt" id="figlgndA1018"><h4 id="A1018"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1018/?report=objectonly" target="object" rid-ob="figobA1018">Figure 6-28</a></h4><p class="float-caption no_bottom_margin">The consensus nucleotide sequences in an RNA molecule that signal the beginning and the end of most introns in humans. Only the three blocks of nucleotide sequences shown are required to remove an intron sequence; the rest of the intron can be occupied <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1018/?report=objectonly" target="object" rid-ob="figobA1018">(more...)</a></p></div></div></div><div id="A1019"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1019_" class="ui-helper-clearfix">RNA Splicing Is Performed by the Spliceosome</h2><p>Unlike the other steps of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> production we have discussed, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> is performed largely by RNA molecules instead of proteins. RNA molecules recognize <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a>-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> borders and participate in the chemistry of splicing. These RNA molecules are relatively short (less than 200 nucleotides each), and there are five of them (U1, U2, U4, U5, and U6) involved in the major form of pre-mRNA splicing. Known as <b>snRNAs (small nuclear RNAs)</b>, each is complexed with at least seven <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> subunits to form a snRNP (small nuclear ribonucleoprotein). These snRNPs form the core of the <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" class="def" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a>, the large assembly of RNA and protein molecules that performs pre-mRNA splicing in the cell.</p><p>The <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> is a dynamic machine; as we see below, it is assembled on pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> from separate components, and parts enter and leave it as the splicing <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> proceeds (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" target="object" rid-figpopup="figA1020" rid-ob="figobA1020">Figure 6-29</a>). During the splicing reaction, recognition of the 5′ splice junction, the branch point site and the 3′ splice junction is performed largely through <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pairing between the snRNAs and the consensus <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> sequences in the pre-mRNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/" role="button" aria-expanded="false" aria-haspopup="true">substrate</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" target="object" rid-figpopup="figA1021" rid-ob="figobA1021">Figure 6-30</a>). In the course of splicing, the spliceosome undergoes several shifts in which one set of base-pair interactions is broken and another is formed in its place. For example, U1 is replaced by U6 at the 5′ splice junction (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" target="object" rid-figpopup="figA1021" rid-ob="figobA1021">Figure 6-30A</a>). As we shall see, this type of RNA-RNA rearrangement (in which the formation of one RNA-RNA interaction requires the disruption of another) occurs several times during the splicing reaction. It permits the checking and rechecking of RNA sequences before the chemical reaction is allowed to proceed, thereby increasing the accuracy of splicing.


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1020" co-legend-rid="figlgndA1020"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" target="object" title="Figure 6-29" class="img_link icnblk_img figpopup" rid-figpopup="figA1020" rid-ob="figobA1020"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f29.gif" src-large="/books/NBK26887/bin/ch6f29.jpg" alt="Figure 6-29. The RNA splicing mechanism."></a><div class="icnblk_cntnt" id="figlgndA1020"><h4 id="A1020"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" target="object" rid-ob="figobA1020">Figure 6-29</a></h4><p class="float-caption no_bottom_margin">The RNA splicing mechanism. RNA splicing is catalyzed by an assembly of snRNPs (shown as <i>colored circles)</i> plus other proteins (most of which are not shown), which together constitute the spliceosome. The spliceosome recognizes the splicing signals on <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" target="object" rid-ob="figobA1020">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1021" co-legend-rid="figlgndA1021"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" target="object" title="Figure 6-30" class="img_link icnblk_img figpopup" rid-figpopup="figA1021" rid-ob="figobA1021"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f30.gif" src-large="/books/NBK26887/bin/ch6f30.jpg" alt="Figure 6-30. Several of the rearrangements that take place in the spliceosome during pre-mRNA splicing."></a><div class="icnblk_cntnt" id="figlgndA1021"><h4 id="A1021"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" target="object" rid-ob="figobA1021">Figure 6-30</a></h4><p class="float-caption no_bottom_margin">Several of the rearrangements that take place in the spliceosome during pre-mRNA splicing. Shown here are the details for the yeast <i>Saccharomyces cerevisiae</i>, in which the nucleotide sequences involved are slightly different from those in human cells. <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" target="object" rid-ob="figobA1021">(more...)</a></p></div></div></div><div id="A1022"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1022_" class="ui-helper-clearfix">The Spliceosome Uses ATP Hydrolysis to Produce a Complex Series of RNA-RNA Rearrangements</h2><p>Although ATP hydrolysis is not required for the chemistry of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> <i>per se</i>, it is required for the stepwise assembly and rearrangements of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a>. Some of the additional proteins that make up the spliceosome are RNA helicases, which use the energy of ATP hydrolysis to break existing RNA-RNA interactions so as to allow the formation of new ones. In fact, all the steps shown previously in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" target="object" rid-figpopup="figA1020" rid-ob="figobA1020">Figure 6-29</a>—except the association of BBP with the branch-point site and U1 snRNP with the 5′ splice site—require ATP hydrolysis and additional proteins. In all, more than 50 proteins, including those that form the snRNPs, are required for each splicing event.</p><p>The ATP-requiring <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>-RNA rearrangements that take place in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> occur within the snRNPs themselves and between the snRNPs and the pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/" role="button" aria-expanded="false" aria-haspopup="true">substrate</a>. One of the most important roles of these rearrangements is the creation of the active catalytic site of the spliceosome. The strategy of creating an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/" role="button" aria-expanded="false" aria-haspopup="true">active site</a> only after the assembly and rearrangement of splicing components on a pre-mRNA substrate is an important way of preventing wayward splicing.</p><p>Perhaps the most surprising feature of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> is the nature of the catalytic site itself: it is largely (if not exclusively) formed by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules instead of proteins. In the last <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5785/" role="button" aria-expanded="false" aria-haspopup="true">section</a> of this chapter we discuss in general terms the structural and chemical properties of RNA that allow it to perform catalysis; here we need only consider that the U2 and U6 snRNAs in the spliceosome form a precise three-dimensional RNA structure that juxtaposes the 5′ splice site of the pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> with the branch-point site and probably performs the first transesterification <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a> (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" target="object" rid-figpopup="figA1021" rid-ob="figobA1021">Figure 6-30C</a>). In a similar way, the 5′ and 3′ splice junctions are brought together (an event requiring the U5 <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5812/" role="button" aria-expanded="false" aria-haspopup="true">snRNA</a>) to facilitate the second transesterification.</p><p>Once the splicing chemistry is completed, the snRNPs remain bound to the lariat and the spliced product is released. The disassembly of these snRNPs from the lariat (and from each other) requires another series of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>-RNA rearrangements that require ATP hydrolysis, thereby returning the snRNAs to their original configuration so that they can be used again in a new <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a>.</p></div><div id="A1023"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1023_" class="ui-helper-clearfix">Ordering Influences in the Pre-mRNA Help to Explain How the Proper Splice Sites Are Chosen</h2><p>As we have seen, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences vary enormously in size, with some being in excess of 100,000 nucleotides. If splice-site selection were <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/" role="button" aria-expanded="false" aria-haspopup="true">determined</a> solely by the snRNPs acting on a preformed, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>-free <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a>, we would expect splicing mistakes—such as <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> skipping and the use of cryptic splice sites—to be very common (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1024/?report=objectonly" target="object" rid-figpopup="figA1024" rid-ob="figobA1024">Figure 6-31</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1024" co-legend-rid="figlgndA1024"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1024/?report=objectonly" target="object" title="Figure 6-31" class="img_link icnblk_img figpopup" rid-figpopup="figA1024" rid-ob="figobA1024"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f31.gif" src-large="/books/NBK26887/bin/ch6f31.jpg" alt="Figure 6-31. Two types of splicing errors."></a><div class="icnblk_cntnt" id="figlgndA1024"><h4 id="A1024"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1024/?report=objectonly" target="object" rid-ob="figobA1024">Figure 6-31</a></h4><p class="float-caption no_bottom_margin">Two types of splicing errors. Both types might be expected to occur frequently if splice-site selection were performed by the spliceosome on a preformed, protein-free RNA molecule. “Cryptic” splicing signals are nucleotide sequences of <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1024/?report=objectonly" target="object" rid-ob="figobA1024">(more...)</a></p></div></div><p>The fidelity mechanisms built into the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> are supplemented by two additional factors that help ensure that splicing occurs accurately. These ordering influences in the pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> increase the probability that the appropriate pairs of 5′ and 3′ splice sites will be brought together in the spliceosome before the splicing chemistry begins. The first results from the assembly of the spliceosome occurring as the pre-mRNA emerges from a transcribing <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1010/?report=objectonly" target="object" rid-figpopup="figA1010" rid-ob="figobA1010">Figure 6-23</a>). As for 5′ cap formation, several components of the spliceosome seem to be carried on the phosphorylated tail of RNA polymerase. Their transfer directly from the polymerase to the nascent pre-mRNA presumably helps the cell to keep track of introns and exons: the snRNPs at a 5′ splice site are initially presented with only a single 3′ splice site since the sites further downstream have not yet been synthesized. This feature helps to prevent inappropriate <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> skipping.</p><p>The second factor that helps the cell to choose splice sites has been termed the “<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> definition hypothesis,” and it is understood only in outline. Exon size tends to be much more uniform than <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> size, averaging about 150 <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> pairs across a wide variety of eucaryotic organisms (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1025/?report=objectonly" target="object" rid-figpopup="figA1025" rid-ob="figobA1025">Figure 6-32</a>). As <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> synthesis proceeds, a group of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> components, called the SR proteins (so-named because they contain a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/" role="button" aria-expanded="false" aria-haspopup="true">domain</a> rich in serines and arginines), are thought to assemble on exon sequences and mark off each 3′ and 5′ splice site starting at the 5′ end of the RNA (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1026/?report=objectonly" target="object" rid-figpopup="figA1026" rid-ob="figobA1026">Figure 6-33</a>). This assembly takes place in conjunction with the U1 <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5812/" role="button" aria-expanded="false" aria-haspopup="true">snRNA</a>, which marks one exon boundary, and U2AF, which initially helps to specify the other. By specifically marking the exons in this way, the cell increases the accuracy with which the initial splicing components are deposited on the nascent RNA and thereby helps to avoid cryptic splice sites. How the SR proteins discriminate exon sequences from intron sequences is not understood; however, it is known that some of the SR proteins bind preferentially to RNA sequences in specific exons. In principle, the redundancy in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5218/" role="button" aria-expanded="false" aria-haspopup="true">genetic code</a> could have been exploited during evolution to select for binding sites for SR proteins in exons, allowing these sites to be created without constraining <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/" role="button" aria-expanded="false" aria-haspopup="true">amino acid</a> sequences.


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1025" co-legend-rid="figlgndA1025"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1025/?report=objectonly" target="object" title="Figure 6-32" class="img_link icnblk_img figpopup" rid-figpopup="figA1025" rid-ob="figobA1025"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f32.gif" src-large="/books/NBK26887/bin/ch6f32.jpg" alt="Figure 6-32. Variation in intron and exon lengths in the human, worm, and fly genomes."></a><div class="icnblk_cntnt" id="figlgndA1025"><h4 id="A1025"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1025/?report=objectonly" target="object" rid-ob="figobA1025">Figure 6-32</a></h4><p class="float-caption no_bottom_margin">Variation in intron and exon lengths in the human, worm, and fly genomes. (A) Size distribution of exons. (B) Size distribution of introns. Note that exon length is much more uniform than intron length. (Adapted from International Human Genome Sequencing <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1025/?report=objectonly" target="object" rid-ob="figobA1025">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1026" co-legend-rid="figlgndA1026"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1026/?report=objectonly" target="object" title="Figure 6-33" class="img_link icnblk_img figpopup" rid-figpopup="figA1026" rid-ob="figobA1026"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f33.gif" src-large="/books/NBK26887/bin/ch6f33.jpg" alt="Figure 6-33. The exon definition hypothesis."></a><div class="icnblk_cntnt" id="figlgndA1026"><h4 id="A1026"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1026/?report=objectonly" target="object" rid-ob="figobA1026">Figure 6-33</a></h4><p class="float-caption no_bottom_margin">The exon definition hypothesis. According to one proposal, SR proteins bind to each exon sequence in the pre-mRNA and thereby help to guide the snRNPs to the proper intron/exon boundaries. This demarcation of exons by the SR proteins occurs co-transcriptionally, <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1026/?report=objectonly" target="object" rid-ob="figobA1026">(more...)</a></p></div></div><p>Both the marking out of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> boundaries and the assembly of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> begin on an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> while it is still being elongated by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> at its 3′ end. However, the actual chemistry of splicing can take place much later. This delay means that intron sequences are not necessarily removed from a pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> molecule in the order in which they occur along the RNA chain. It also means that, although spliceosome assembly is co-transcriptional, the splicing reactions sometimes occur posttranscriptionally—that is, after a complete pre-mRNA molecule has been made.</p></div><div id="A1027"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1027_" class="ui-helper-clearfix">A Second Set of snRNPs Splice a Small Fraction of Intron Sequences in Animals and Plants</h2><p>Simple eucaryotes such as <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5937/" role="button" aria-expanded="false" aria-haspopup="true">yeast</a> have only one set of snRNPs that perform all pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing. However, more <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> eucaryotes such as flies, mammals, and plants have a second set of snRNPs that direct the splicing of a small fraction of their <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences. This minor form of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> recognizes a different set of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> sequences at the 5′ and 3′ splice junctions and at the branch point; it is called the <i>AT-AC spliceosome</i> because of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence determinants at its intron-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> borders (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" rid-figpopup="figA1028" rid-ob="figobA1028">Figure 6-34</a>). Despite recognizing different nucleotide sequences, the snRNPs in this spliceosome make the same types of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>-RNA interactions with the pre-mRNA and with each other as do the major snRNPs (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" rid-figpopup="figA1028" rid-ob="figobA1028">Figure 6-34B</a>). The recent discovery of this class of snRNPs gives us confidence in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pair interactions deduced for the major spliceosome, because it provides an independent set of molecules that undergo the same RNA-RNA interactions despite differences in the RNA sequences involved.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1028" co-legend-rid="figlgndA1028"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" title="Figure 6-34" class="img_link icnblk_img figpopup" rid-figpopup="figA1028" rid-ob="figobA1028"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f34.gif" src-large="/books/NBK26887/bin/ch6f34.jpg" alt="Figure 6-34. Outline of the mechanisms used for three types of RNA splicing."></a><div class="icnblk_cntnt" id="figlgndA1028"><h4 id="A1028"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" rid-ob="figobA1028">Figure 6-34</a></h4><p class="float-caption no_bottom_margin">Outline of the mechanisms used for three types of RNA splicing. (A) Three types of spliceosomes. The major spliceosome <i>(left),</i> the AT-AC spliceosome (middle), and the trans-spliceosome <i>(right)</i> are each shown at two stages of assembly. The U5 snRNP is <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" rid-ob="figobA1028">(more...)</a></p></div></div><p>A particular variation on splicing, called <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5897/" class="def" role="button" aria-expanded="false" aria-haspopup="true">trans-splicing</a>, has been discovered in a few eucaryotic organisms. These include the single-celled trypanosomes—protozoans that cause African sleeping sickness in humans—and the model multicellular organism, the nematode worm. In <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5897/" role="button" aria-expanded="false" aria-haspopup="true">trans-splicing</a>, exons from two separate <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> transcripts are spliced together to form a mature <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" rid-figpopup="figA1028" rid-ob="figobA1028">Figure 6-34</a>). Trypanosomes produce all of their mRNAs in this way, whereas only about 1% of nematode mRNAs are produced by trans-splicing. In both cases, a single <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> is spliced onto the 5′ end of many different RNA transcripts produced by the cell; in this way, all of the products of trans-splicing have the same 5′ exon and different 3′ exons. Many of the same snRNPs that function in conventional splicing are used in this <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a>, although trans-splicing uses a unique snRNP (called the SL RNP) that brings in the common exon (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1028/?report=objectonly" target="object" rid-figpopup="figA1028" rid-ob="figobA1028">Figure 6-34</a>).</p><p>The reason that a few organisms use <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5897/" role="button" aria-expanded="false" aria-haspopup="true">trans-splicing</a> is not known; however, it is thought that the common 5′ <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> may aid in the translation of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a>. Thus, the products of trans-splicing in nematodes seem to be translated with especially high efficiency.</p></div><div id="A1029"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1029_" class="ui-helper-clearfix">RNA Splicing Shows Remarkable Plasticity</h2><p>We have seen that the choice of splice sites depends on many features of the pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a>; these include the affinity of the three signals on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> (the 5′ and 3′ splice junctions and branch point) for the splicing machinery, the length and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a>, the co-transcriptional assembly of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a>, and the accuracy of the “bookkeeping” that underlies exon definition. So far we have emphasized the accuracy of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> processes that occur in a cell. But it also seems that the mechanism has been selected for its flexibility, which allows the cell to try out new proteins on occasion. Thus, for example, when a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5502/" role="button" aria-expanded="false" aria-haspopup="true">mutation</a> occurs in a nucleotide sequence critical for splicing of a particular <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a>, it does not necessarily prevent splicing of that intron altogether. Instead, the mutation typically creates a new pattern of splicing (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1030/?report=objectonly" target="object" rid-figpopup="figA1030" rid-ob="figobA1030">Figure 6-35</a>). Most commonly, an exon is simply skipped (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1030/?report=objectonly" target="object" rid-figpopup="figA1030" rid-ob="figobA1030">Figure 6-35B</a>). In other cases, the mutation causes a “cryptic” splice junction to be used (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1030/?report=objectonly" target="object" rid-figpopup="figA1030" rid-ob="figobA1030">Figure 6-35C</a>). Presumably, the splicing machinery has evolved to pick out the best possible pattern of splice junctions, and if the optimal one is damaged by mutation, it will seek out the next best pattern and so on. This flexibility in the process of RNA splicing suggests that changes in splicing patterns caused by random mutations have been an important pathway in the evolution of genes and organisms.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1030" co-legend-rid="figlgndA1030"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1030/?report=objectonly" target="object" title="Figure 6-35" class="img_link icnblk_img figpopup" rid-figpopup="figA1030" rid-ob="figobA1030"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f35.gif" src-large="/books/NBK26887/bin/ch6f35.jpg" alt="Figure 6-35. Abnormal processing of the β-globin primary RNA transcript in humans with the disease β thalassemia."></a><div class="icnblk_cntnt" id="figlgndA1030"><h4 id="A1030"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1030/?report=objectonly" target="object" rid-ob="figobA1030">Figure 6-35</a></h4><p class="float-caption no_bottom_margin">Abnormal processing of the β-globin primary RNA transcript in humans with the disease β thalassemia.  In the examples shown, the disease is caused by splice-site mutations, denoted by <i>black arrowheads</i>. The <i>dark blue boxes</i> represent the <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1030/?report=objectonly" target="object" rid-ob="figobA1030">(more...)</a></p></div></div><p>The plasticity of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> also means that the cell can easily regulate the pattern of RNA splicing. Earlier in this <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5785/" role="button" aria-expanded="false" aria-haspopup="true">section</a> we saw that alternative splicing can give rise to different proteins from the same <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>. Some examples of alternative splicing are <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5025/" role="button" aria-expanded="false" aria-haspopup="true">constitutive</a>; that is, the alternatively spliced mRNAs are produced continuously by cells of an organism. However, in most cases, the splicing patterns are regulated by the cell so that different forms of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> are produced at different times and in different tissues (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1016/?report=objectonly" target="object" rid-figpopup="figA1016" rid-ob="figobA1016">Figure 6-27</a>). In Chapter 7 we return to this issue to discuss some specific examples of regulated RNA splicing.</p></div><div id="A1031"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1031_" class="ui-helper-clearfix">Spliceosome-catalyzed RNA Splicing Probably Evolved from Self-splicing Mechanisms</h2><p>When the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> was first discovered, it puzzled molecular biologists. Why do <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules instead of proteins perform important roles in splice site recognition and in the chemistry of splicing? Why is a lariat intermediate used rather than the apparently simpler alternative of bringing the 5′ and 3′ splice sites together in a single step, followed by their direct <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/" role="button" aria-expanded="false" aria-haspopup="true">cleavage</a> and rejoining? The answers to these questions reflect the way in which the spliceosome is believed to have evolved.</p><p>As discussed briefly in Chapter 1 (and taken up again in more detail in the final <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5785/" role="button" aria-expanded="false" aria-haspopup="true">section</a> of this chapter), it is thought that early cells used <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules rather than proteins as their major catalysts and that they stored their genetic information in RNA rather than in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> sequences. RNA-catalyzed splicing reactions presumably had important roles in these early cells. As evidence, some <i>self-splicing RNA</i> introns (that is, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences in RNA whose splicing out can occur in the absence of proteins or any other RNA molecules) remain today—for example, in the nuclear <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a> genes of the ciliate <i>Tetrahymena</i>, in a few bacteriophage T4 genes, and in some mitochondrial and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4969/" role="button" aria-expanded="false" aria-haspopup="true">chloroplast</a> genes.</p><p>A self-splicing <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequence can be identified in a test tube by incubating a pure <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> that contains the intron sequence and observing the splicing <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/" role="button" aria-expanded="false" aria-haspopup="true">reaction</a>. Two major classes of self-splicing intron sequences can be distinguished in this way. <i>Group I intron sequences</i> begin the splicing reaction by binding a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/" role="button" aria-expanded="false" aria-haspopup="true">G</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> to the intron sequence; this G is thereby activated to form the attacking group that will break the first of the phosphodiester bonds cleaved during splicing (the bond at the 5′ splice site). In <i>group II intron sequences,</i> an especially reactive A <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5737/" role="button" aria-expanded="false" aria-haspopup="true">residue</a> in the intron sequence is the attacking group, and a lariat intermediate is generated. Otherwise the reaction pathways for the two types of self-splicing intron sequences are the same. Both are presumed to represent vestiges of very ancient mechanisms (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1032/?report=objectonly" target="object" rid-figpopup="figA1032" rid-ob="figobA1032">Figure 6-36</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1032" co-legend-rid="figlgndA1032"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1032/?report=objectonly" target="object" title="Figure 6-36" class="img_link icnblk_img figpopup" rid-figpopup="figA1032" rid-ob="figobA1032"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f36.gif" src-large="/books/NBK26887/bin/ch6f36.jpg" alt="Figure 6-36. The two known classes of self-splicing intron sequences."></a><div class="icnblk_cntnt" id="figlgndA1032"><h4 id="A1032"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1032/?report=objectonly" target="object" rid-ob="figobA1032">Figure 6-36</a></h4><p class="float-caption no_bottom_margin">The two known classes of self-splicing intron sequences. The group I intron sequences bind a free G nucleotide to a specific site on the RNA to initiate splicing, while the group II intron sequences use an especially reactive A nucleotide in the intron <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1032/?report=objectonly" target="object" rid-ob="figobA1032">(more...)</a></p></div></div><p>For both types of self-splicing reactions, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequence of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> is critical; the intron <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> folds into a specific three-dimensional structure, which brings the 5′ and 3′ splice junctions together and provides precisely positioned reactive groups to perform the chemistry (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A980/?report=objectonly" target="object" rid-figpopup="figA980" rid-ob="figobA980">Figure 6-6C</a>). Based on the fact that the chemistries of their splicing reactions are so similar, it has been proposed that the pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing mechanism of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> evolved from group II splicing. According to this idea, when the spliceosomal snRNPs took over the structural and chemical roles of the group II introns, the strict sequence constraints on intron sequences would have disappeared, thereby permitting a vast expansion in the number of different RNAs that could be spliced.</p></div><div id="A1033"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1033_" class="ui-helper-clearfix">RNA-Processing Enzymes Generate the 3′ End of Eucaryotic mRNAs</h2><p>As previously explained, the 5′ end of the pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> produced by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II is capped almost as soon as it emerges from the RNA polymerase. Then, as the polymerase continues its movement along a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> components assemble on the RNA and delineate the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> boundaries. The long C-terminal tail of the RNA polymerase coordinates these processes by transferring capping and splicing components directly to the RNA as the RNA emerges from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a>. As we see in this <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5785/" role="button" aria-expanded="false" aria-haspopup="true">section</a>, as RNA polymerase II terminates transcription at the end of a gene, it uses a similar mechanism to ensure that the 3′ end of the pre-mRNA becomes appropriately processed.</p><p>As might be expected, the 3′ ends of mRNAs are ultimately specified by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> signals encoded in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1034/?report=objectonly" target="object" rid-figpopup="figA1034" rid-ob="figobA1034">Figure 6-37</a>). These DNA signals are transcribed into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> as the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II moves through them, and they are then recognized (as RNA) by a series of RNA-binding proteins and RNA-processing enzymes (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1035/?report=objectonly" target="object" rid-figpopup="figA1035" rid-ob="figobA1035">Figure 6-38</a>). Two multisubunit proteins, called CstF (<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/" role="button" aria-expanded="false" aria-haspopup="true">cleavage</a> stimulation factor F) and CPSF (cleavage and polyadenylation specificity factor), are of special importance. Both of these proteins travel with the RNA polymerase tail and are transferred to the 3′ end processing sequence on an RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> as it emerges from the RNA polymerase. Some of the subunits of CPSF are associated with the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5217/" role="button" aria-expanded="false" aria-haspopup="true">general transcription factor</a> TFIID, which, as we saw earlier in this chapter, is involved in transcription initiation. During transcription initiation, these subunits may be transferred from TFIID to the RNA polymerase tail, remaining associated there until the polymerase has transcribed through the end of a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a>.


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1034" co-legend-rid="figlgndA1034"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1034/?report=objectonly" target="object" title="Figure 6-37" class="img_link icnblk_img figpopup" rid-figpopup="figA1034" rid-ob="figobA1034"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f37.gif" src-large="/books/NBK26887/bin/ch6f37.jpg" alt="Figure 6-37. Consensus nucleotide sequences that direct cleavage and polyadenylation to form the 3′ end of a eucaryotic mRNA."></a><div class="icnblk_cntnt" id="figlgndA1034"><h4 id="A1034"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1034/?report=objectonly" target="object" rid-ob="figobA1034">Figure 6-37</a></h4><p class="float-caption no_bottom_margin">Consensus nucleotide sequences that direct cleavage and polyadenylation to form the 3′ end of a eucaryotic mRNA.  These sequences are encoded in the genome and are recognized by specific proteins after they are transcribed into RNA. The hexamer <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1034/?report=objectonly" target="object" rid-ob="figobA1034">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1035" co-legend-rid="figlgndA1035"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1035/?report=objectonly" target="object" title="Figure 6-38" class="img_link icnblk_img figpopup" rid-figpopup="figA1035" rid-ob="figobA1035"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f38.gif" src-large="/books/NBK26887/bin/ch6f38.jpg" alt="Figure 6-38. Some of the major steps in generating the 3′ end of a eucaryotic mRNA."></a><div class="icnblk_cntnt" id="figlgndA1035"><h4 id="A1035"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1035/?report=objectonly" target="object" rid-ob="figobA1035">Figure 6-38</a></h4><p class="float-caption no_bottom_margin">Some of the major steps in generating the 3′ end of a eucaryotic mRNA.  This process is much more complicated than the analogous process in bacteria, where the RNA polymerase simply stops at a termination signal and releases both the 3′ <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1035/?report=objectonly" target="object" rid-ob="figobA1035">(more...)</a></p></div></div><p>Once CstF and CPSF bind to specific <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequences on an emerging <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a>, additional proteins assemble with them to perform the processing that creates the 3′ end of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a>. First, the RNA is cleaved (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1035/?report=objectonly" target="object" rid-figpopup="figA1035" rid-ob="figobA1035">Figure 6-38</a>). Next an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> called poly-A polymerase adds, one at a time, approximately 200 A nucleotides to the 3′ end produced by the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/" role="button" aria-expanded="false" aria-haspopup="true">cleavage</a>. The nucleotide precursor for these additions is ATP, and the same type of 5′-to-3′ bonds are formed as in conventional RNA synthesis (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A978/?report=objectonly" target="object" rid-figpopup="figA978" rid-ob="figobA978">Figure 6-4</a>). Unlike the usual RNA polymerases, poly-A polymerase does not require a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a>; hence the poly-A tail of eucaryotic mRNAs is not directly encoded in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a>. As the poly-A tail is synthesized, proteins called poly-A-binding proteins assemble onto it and, by a poorly understood mechanism, determine the final length of the tail. Poly-A-binding proteins remain bound to the poly-A tail as the mRNA makes its journey from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a> to the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5055/" role="button" aria-expanded="false" aria-haspopup="true">cytosol</a> and they help to direct the synthesis of a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/" role="button" aria-expanded="false" aria-haspopup="true">ribosome</a>, as we see later in this chapter.</p><p>After the 3′ end of a eucaryotic pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> has been cleaved, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II continues to transcribe, in some cases continuing as many as several hundred nucleotides beyond the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> that contains the 3′ <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/" role="button" aria-expanded="false" aria-haspopup="true">cleavage</a>-site information. But the polymerase soon releases its grip on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> and transcription terminates; the piece of RNA downstream of the cleavage site is then degraded in the cell <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>. It is not yet understood what triggers the loss in polymerase II processivity after the RNA is cleaved. One idea is that the transfer of the 3′ end processing factors from the RNA polymerase to the RNA causes a conformational change in the polymerase that loosens its hold on DNA; another is that the lack of a cap structure (and the CBC) on the 5′ end of the RNA that emerges from the polymerase somehow signals to the polymerase to terminate transcription.</p></div><div id="A1036"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1036_" class="ui-helper-clearfix">Mature Eucaryotic mRNAs Are Selectively Exported from the Nucleus</h2><p>We have seen how eucaryotic pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> synthesis and processing takes place in an orderly fashion within the cell <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>. However, these events create a special problem for eucaryotic cells, especially those of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> organisms where the introns are vastly longer than the exons. Of the pre-mRNA that is synthesized, only a small fraction—the mature mRNA—is of further use to the cell. The rest—excised introns, broken RNAs, and aberrantly spliced pre-mRNAs—is not only useless but could be dangerous if it was not destroyed. How then does the cell distinguish between the relatively rare mature mRNA molecules it wishes to keep and the overwhelming amount of debris from <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> processing? The answer is that transport of mRNA from the nucleus to the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5053/" role="button" aria-expanded="false" aria-haspopup="true">cytoplasm</a>, where it is translated into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>, is highly selective—being closely coupled to correct RNA processing. This coupling is achieved by the <i><a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5557/" role="button" aria-expanded="false" aria-haspopup="true">nuclear pore complex</a>,</i> which recognizes and transports only completed mRNAs.</p><p>We have seen that as a pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> is synthesized and processed, it is bound by a variety of proteins, including the cap-binding <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a>, the SR proteins, and the poly-A binding proteins. To be “export-ready,” it seems than an mRNA must be bound by the appropriate set of proteins—with certain proteins such as the cap-binding complex being present, and others such as snRNP proteins absent. Additional proteins, placed on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> during splicing, seem to mark <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a>-exon boundaries and thereby signify completed splicing events. Only if the proper set of proteins is bound to an mRNA is it guided through the <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5557/" class="def" role="button" aria-expanded="false" aria-haspopup="true">nuclear pore complex</a> into the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5055/" role="button" aria-expanded="false" aria-haspopup="true">cytosol</a>. As described in Chapter 12, nuclear pore complexes are <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4840/" role="button" aria-expanded="false" aria-haspopup="true">aqueous</a> channels in the nuclear <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5438/" role="button" aria-expanded="false" aria-haspopup="true">membrane</a> that directly connect the nucleoplasm and cytosol. Small molecules (less than 50,000 daltons) can diffuse freely through them. However, most of the macromolecules in cells, including mRNAs complexed with proteins, are far too large to pass through the pores without a special process to move them. An <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4773/" role="button" aria-expanded="false" aria-haspopup="true">active transport</a> of substances through the nuclear pore complexes occurs in both directions. As explained in Chapter 12, signals on the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5419/" role="button" aria-expanded="false" aria-haspopup="true">macromolecule</a> determine whether it is exported from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a> (a mRNA, for example) or imported into it (an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a>, for example). For the case of mRNAs, the bound proteins that mark completed splicing events are of particular importance, as they are known to serve directly as RNA export factors (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A2154/figure/A2167/?report=objectonly" target="object">Figure 12-16</a>). mRNAs transcribed from genes that lack introns apparently contain <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> sequences that are directly recognized by other RNA export factors. Eucaryotic cells thus use their nuclear pore complexes as gates that allow only useful RNA molecules to enter the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5053/" role="button" aria-expanded="false" aria-haspopup="true">cytoplasm</a>.</p><p>Of all the proteins that assemble on pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> molecules as they emerge from transcribing <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> polymerases, the most abundant are the hnRNPs (heterogeneous nuclear ribonuclear proteins). Some of these proteins (there are approximately 30 of them in humans) remove the hairpin helices from the RNA so that splicing and other signals on the RNA can be read more easily. Others package the RNA contained in the very long <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences typically found in genes of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> organisms (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1026/?report=objectonly" target="object" rid-figpopup="figA1026" rid-ob="figobA1026">Figure 6-33</a>). Apart from histones, certain hnRNP proteins are the most abundant proteins in the cell <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>, and they may play a particularly important role in distinguishing mature mRNA from processing debris. hnRNP particles (<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5566/" role="button" aria-expanded="false" aria-haspopup="true">nucleosome</a>-like complexes of hnRNP proteins and RNA—see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1026/?report=objectonly" target="object" rid-figpopup="figA1026" rid-ob="figobA1026">Figure 6-33</a>) are largely excluded from <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/" role="button" aria-expanded="false" aria-haspopup="true">exon</a> sequences, perhaps by prior binding of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/" role="button" aria-expanded="false" aria-haspopup="true">spliceosome</a> components. They remain on excised introns and probably help mark them for nuclear retention and eventual destruction.</p><p>The export of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a>-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> complexes from the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a> can be observed with an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5122/" role="button" aria-expanded="false" aria-haspopup="true">electron microscope</a> for the unusually abundant mRNA of the insect Balbiani Ring genes. As these genes are transcribed, the newly formed <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> is seen to be packaged by proteins (including hnRNP and SR proteins). This protein-RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a> undergoes a series of structural transitions, probably reflecting RNA processing events, culminating in a curved fiber (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1037/?report=objectonly" target="object" rid-figpopup="figA1037" rid-ob="figobA1037">Figure 6-39</a>). This curved fiber then moves through the nucleoplasm and enters the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5557/" role="button" aria-expanded="false" aria-haspopup="true">nuclear pore complex</a> (with its 5′ cap proceeding first), and it undergoes another series of structural transitions as it moves through the NPC. These and other observations reveal that the pre-mRNA-protein and mRNA-protein complexes are dynamic structures that gain and lose numerous specific proteins during RNA synthesis, processing, export, and translation (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1038/?report=objectonly" target="object" rid-figpopup="figA1038" rid-ob="figobA1038">Figure 6-40</a>).


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1037" co-legend-rid="figlgndA1037"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1037/?report=objectonly" target="object" title="Figure 6-39" class="img_link icnblk_img figpopup" rid-figpopup="figA1037" rid-ob="figobA1037"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f39.gif" src-large="/books/NBK26887/bin/ch6f39.jpg" alt="Figure 6-39. Transport of a large mRNA molecule through the nuclear pore complex."></a><div class="icnblk_cntnt" id="figlgndA1037"><h4 id="A1037"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1037/?report=objectonly" target="object" rid-ob="figobA1037">Figure 6-39</a></h4><p class="float-caption no_bottom_margin">Transport of a large mRNA molecule through the nuclear pore complex. (A) The maturation of a Balbiani Ring mRNA molecule as it is synthesized by RNA polymerase and packaged by a variety of nuclear proteins. This drawing of unusually abundant RNA produced <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1037/?report=objectonly" target="object" rid-ob="figobA1037">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1038" co-legend-rid="figlgndA1038"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1038/?report=objectonly" target="object" title="Figure 6-40" class="img_link icnblk_img figpopup" rid-figpopup="figA1038" rid-ob="figobA1038"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f40.gif" src-large="/books/NBK26887/bin/ch6f40.jpg" alt="Figure 6-40. Schematic illustration of an “export-ready” mRNA molecule and its transport through the nuclear pore."></a><div class="icnblk_cntnt" id="figlgndA1038"><h4 id="A1038"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1038/?report=objectonly" target="object" rid-ob="figobA1038">Figure 6-40</a></h4><p class="float-caption no_bottom_margin">Schematic illustration of an “export-ready” mRNA molecule and its transport through the nuclear pore. As indicated, some proteins travel with the mRNA as it moves through the pore, whereas others remain in the nucleus. Once in the cytoplasm, <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1038/?report=objectonly" target="object" rid-ob="figobA1038">(more...)</a></p></div></div><p>Before discussing what happens to mRNAs after they leave the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>, we briefly consider how the synthesis and processing of noncoding <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules occurs. Although there are many other examples, our discussion focuses on the rRNAs that are critically important for the translation of mRNAs into <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>.</p></div><div id="A1039"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1039_" class="ui-helper-clearfix">Many Noncoding RNAs Are Also Synthesized and Processed in the Nucleus</h2><p>A few per cent of the dry weight of a mammalian cell is <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>; of that, only about 3–5% is <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a>. A fraction of the remainder represents <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences before they have been degraded, but most of the RNA in cells performs structural and catalytic functions (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/table/A986/?report=objectonly" target="object" rid-figpopup="figA986" rid-ob="figobA986">Table 6-1</a>, p. 306). The most abundant RNAs in cells are the ribosomal RNAs (rRNAs)—constituting approximately 80% of the RNA in rapidly dividing cells. As discussed later in this chapter, these RNAs form the core of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/" role="button" aria-expanded="false" aria-haspopup="true">ribosome</a>. Unlike bacteria—in which all RNAs in the cell are synthesized by a single <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a>—eucaryotes have a separate, specialized polymerase, RNA polymerase I, that is dedicated to producing rRNAs. RNA polymerase I is similar structurally to the RNA polymerase II discussed previously; however, the absence of a C-terminal tail in polymerase I helps to explain why its transcripts are neither capped nor polyadenylated. As discussed earlier, this difference helps the cell distinguish between noncoding RNAs and mRNAs.</p><p>Because multiple rounds of translation of each <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> can provide an enormous amplification in the production of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> molecules, many of the proteins that are very abundant in a cell can be synthesized from genes that are present in a single copy per <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5264/" role="button" aria-expanded="false" aria-haspopup="true">haploid</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/" role="button" aria-expanded="false" aria-haspopup="true">genome</a>. In contrast, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> components of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/" role="button" aria-expanded="false" aria-haspopup="true">ribosome</a> are final <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> products, and a growing mammalian cell must synthesize approximately 10 million copies of each type of ribosomal RNA in each cell generation to construct its 10 million ribosomes. Adequate quantities of ribosomal RNAs can be produced only because the cell contains multiple copies of the <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5768/" class="def" role="button" aria-expanded="false" aria-haspopup="true">rRNA genes</a> that code for ribosomal RNAs <b>(rRNAs)</b>. Even <i>E. coli</i> needs seven copies of its <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a> genes to meet the cell's need for ribosomes. Human cells contain about 200 <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5768/" role="button" aria-expanded="false" aria-haspopup="true">rRNA gene</a> copies per haploid genome, spread out in small clusters on five different chromosomes (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A608/figure/A611/?report=objectonly" target="object">Figure 4-11</a>), while cells of the frog <i>Xenopus</i> contain about 600 rRNA gene copies per haploid genome in a single cluster on one <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4977/" role="button" aria-expanded="false" aria-haspopup="true">chromosome</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1040/?report=objectonly" target="object" rid-figpopup="figA1040" rid-ob="figobA1040">Figure 6-41</a>).

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1040" co-legend-rid="figlgndA1040"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1040/?report=objectonly" target="object" title="Figure 6-41" class="img_link icnblk_img figpopup" rid-figpopup="figA1040" rid-ob="figobA1040"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f41.gif" src-large="/books/NBK26887/bin/ch6f41.jpg" alt="Figure 6-41. Transcription from tandemly arranged rRNA genes, as seen in the electron microscope."></a><div class="icnblk_cntnt" id="figlgndA1040"><h4 id="A1040"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1040/?report=objectonly" target="object" rid-ob="figobA1040">Figure 6-41</a></h4><p class="float-caption no_bottom_margin">Transcription from tandemly arranged rRNA genes, as seen in the electron microscope. The pattern of alternating transcribed gene and nontranscribed spacer is readily seen. A higher-magnification view was shown in Figure 6-9. (From V.E. Foe, <i>Cold Spring</i> <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1040/?report=objectonly" target="object" rid-ob="figobA1040">(more...)</a></p></div></div><p>There are four types of eucaryotic rRNAs, each present in one copy per <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/" role="button" aria-expanded="false" aria-haspopup="true">ribosome</a>. Three of the four rRNAs (18S, 5.8S, and 28S) are made by chemically modifying and cleaving a single large precursor <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1041/?report=objectonly" target="object" rid-figpopup="figA1041" rid-ob="figobA1041">Figure 6-42</a>); the fourth (5S <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>) is synthesized from a separate cluster of genes by a different polymerase, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> III, and does not require chemical modification. It is not known why this one RNA is transcribed separately.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1041" co-legend-rid="figlgndA1041"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1041/?report=objectonly" target="object" title="Figure 6-42" class="img_link icnblk_img figpopup" rid-figpopup="figA1041" rid-ob="figobA1041"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f42.gif" src-large="/books/NBK26887/bin/ch6f42.jpg" alt="Figure 6-42. The chemical modification and nucleolytic processing of a eucaryotic 45S precursor rRNA molecule into three separate ribosomal RNAs."></a><div class="icnblk_cntnt" id="figlgndA1041"><h4 id="A1041"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1041/?report=objectonly" target="object" rid-ob="figobA1041">Figure 6-42</a></h4><p class="float-caption no_bottom_margin">The chemical modification and nucleolytic processing of a eucaryotic 45S precursor rRNA molecule into three separate ribosomal RNAs. As indicated, two types of chemical modifications (shown in Figure 6-43) are made to the precursor rRNA before it is cleaved. <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1041/?report=objectonly" target="object" rid-ob="figobA1041">(more...)</a></p></div></div><p>Extensive chemical modifications occur in the 13,000-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a>-long precursor <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a> before the rRNAs are cleaved out of it and assembled into ribosomes. These include about 100 methylations of the 2′-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5308/" role="button" aria-expanded="false" aria-haspopup="true">OH</a> positions on nucleotide sugars and 100 isomerizations of uridine nucleotides to pseudouridine (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1042/?report=objectonly" target="object" rid-figpopup="figA1042" rid-ob="figobA1042">Figure 6-43A</a>). The functions of these modifications are not understood in detail, but they probably aid in the folding and assembly of the final rRNAs and may also subtly alter the function of ribosomes. Each modification is made at a specific position in the precursor rRNA. These positions are specified by several hundred “guide RNAs,” which locate themselves through <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/" role="button" aria-expanded="false" aria-haspopup="true">base</a>-pairing to the precursor rRNA and thereby bring an <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>-modifying <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> to the appropriate position (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1042/?report=objectonly" target="object" rid-figpopup="figA1042" rid-ob="figobA1042">Figure 6-43B</a>). Other guide RNAs promote <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/" role="button" aria-expanded="false" aria-haspopup="true">cleavage</a> of the precursor rRNAs into the mature rRNAs, probably by causing conformational changes in the precursor rRNA. All of these guide RNAs are members of a large class of RNAs called <b>small nucleolar RNAs</b> (or <b>snoRNAs</b>), so named because these RNAs perform their functions in a subcompartment of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a> called the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a>. Many snoRNAs are encoded in the introns of other genes, especially those encoding ribosomal proteins. They are therefore synthesized by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II and processed from excised <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1042" co-legend-rid="figlgndA1042"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1042/?report=objectonly" target="object" title="Figure 6-43" class="img_link icnblk_img figpopup" rid-figpopup="figA1042" rid-ob="figobA1042"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f43.gif" src-large="/books/NBK26887/bin/ch6f43.jpg" alt="Figure 6-43. Modifications of the precursor rRNA by guide RNAs."></a><div class="icnblk_cntnt" id="figlgndA1042"><h4 id="A1042"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1042/?report=objectonly" target="object" rid-ob="figobA1042">Figure 6-43</a></h4><p class="float-caption no_bottom_margin">Modifications of the precursor rRNA by guide RNAs. (A) Two prominent covalent modifications occur after rRNA synthesis; the differences from the initially incorporated nucleotide are indicated by <i>red</i> atoms. (B) As indicated, snoRNAs locate the sites of <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1042/?report=objectonly" target="object" rid-ob="figobA1042">(more...)</a></p></div></div></div><div id="A1043"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1043_" class="ui-helper-clearfix">The Nucleolus Is a Ribosome-Producing Factory</h2><p>The <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a> is the most obvious structure seen in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a> of a eucaryotic cell when viewed in the light microscope. Consequently, it was so closely scrutinized by early cytologists that an 1898 review could list some 700 references. We now know that the nucleolus is the site for the processing of rRNAs and their assembly into ribosomes. Unlike other organelles in the cell, it is not bound by a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5438/" role="button" aria-expanded="false" aria-haspopup="true">membrane</a> (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1044/?report=objectonly" target="object" rid-figpopup="figA1044" rid-ob="figobA1044">Figure 6-44</a>); instead, it is a large aggregate of macromolecules, including the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a> genes themselves, precursor rRNAs, mature rRNAs, rRNA-processing enzymes, snoRNPs, ribosomal <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> subunits and partly assembled ribosomes. The close association of all these components presumably allows the assembly of ribosomes to occur rapidly and smoothly.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1044" co-legend-rid="figlgndA1044"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1044/?report=objectonly" target="object" title="Figure 6-44" class="img_link icnblk_img figpopup" rid-figpopup="figA1044" rid-ob="figobA1044"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f44.gif" src-large="/books/NBK26887/bin/ch6f44.jpg" alt="Figure 6-44. Electron micrograph of a thin section of a nucleolus in a human fibroblast, showing its three distinct zones."></a><div class="icnblk_cntnt" id="figlgndA1044"><h4 id="A1044"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1044/?report=objectonly" target="object" rid-ob="figobA1044">Figure 6-44</a></h4><p class="float-caption no_bottom_margin">Electron micrograph of a thin section of a nucleolus in a human fibroblast, showing its three distinct zones. (A) View of entire nucleus. (B) High-power view of the nucleolus. It is believed that transcription of the rRNA genes takes place between the <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1044/?report=objectonly" target="object" rid-ob="figobA1044">(more...)</a></p></div></div><p>It is not yet understood how the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a> is held together and organized, but various types of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> molecules play a central part in its chemistry and structure, suggesting that the nucleolus may have evolved from an ancient structure present in cells dominated by RNA catalysis. In present-day cells, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/" role="button" aria-expanded="false" aria-haspopup="true">rRNA</a> genes also have an important role in forming the nucleolus. In a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5077/" role="button" aria-expanded="false" aria-haspopup="true">diploid</a> human cell, the rRNA genes are distributed into 10 clusters, each of which is located near the tip of one of the two copies of five different chromosomes (see <a href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A608/figure/A611/?report=objectonly" target="object">Figure 4-11</a>). Each time a human cell undergoes <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5477/" role="button" aria-expanded="false" aria-haspopup="true">mitosis</a>, the chromosomes disperse and the nucleolus breaks up; after mitosis, the tips of the 10 chromosomes coalesce as the nucleolus reforms (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1045/?report=objectonly" target="object" rid-figpopup="figA1045" rid-ob="figobA1045">Figures 6-45</a> and <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1046/?report=objectonly" target="object" rid-figpopup="figA1046" rid-ob="figobA1046">6-46</a>). The transcription of the rRNA genes by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> I is necessary for this process.


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1045" co-legend-rid="figlgndA1045"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1045/?report=objectonly" target="object" title="Figure 6-45" class="img_link icnblk_img figpopup" rid-figpopup="figA1045" rid-ob="figobA1045"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f45.gif" src-large="/books/NBK26887/bin/ch6f45.jpg" alt="Figure 6-45. Changes in the appearance of the nucleolus in a human cell during the cell cycle."></a><div class="icnblk_cntnt" id="figlgndA1045"><h4 id="A1045"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1045/?report=objectonly" target="object" rid-ob="figobA1045">Figure 6-45</a></h4><p class="float-caption no_bottom_margin">Changes in the appearance of the nucleolus in a human cell during the cell cycle. Only the cell nucleus is represented in this diagram. In most eucaryotic cells the nuclear membrane breaks down during mitosis, as indicated by the dashed circles. </p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1046" co-legend-rid="figlgndA1046"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1046/?report=objectonly" target="object" title="Figure 6-46" class="img_link icnblk_img figpopup" rid-figpopup="figA1046" rid-ob="figobA1046"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f46.gif" src-large="/books/NBK26887/bin/ch6f46.jpg" alt="Figure 6-46. Nucleolar fusion."></a><div class="icnblk_cntnt" id="figlgndA1046"><h4 id="A1046"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1046/?report=objectonly" target="object" rid-ob="figobA1046">Figure 6-46</a></h4><p class="float-caption no_bottom_margin">Nucleolar fusion. These light micrographs of human fibroblasts grown in culture show various stages of nucleolar fusion. After mitosis, each of the ten human chromosomes that carry a cluster of rRNA genes begins to form a tiny nucleolus, but these rapidly <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1046/?report=objectonly" target="object" rid-ob="figobA1046">(more...)</a></p></div></div><p>As might be expected, the size of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a> reflects the number of ribosomes that the cell is producing. Its size therefore varies greatly in different cells and can change in a single cell, occupying 25% of the total nuclear volume in cells that are making unusually large amounts of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>.</p><p>A schematic diagram of the assembly of ribosomes is shown in <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1047/?report=objectonly" target="object" rid-figpopup="figA1047" rid-ob="figobA1047">Figure 6-47</a>. In addition to its important role in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/" role="button" aria-expanded="false" aria-haspopup="true">ribosome</a> biogenesis, the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a> is also the site where other RNAs are produced and other <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a>-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> complexes are assembled. For example, the U6 snRNP, which, as we have seen, functions in pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing (see <a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" target="object" rid-figpopup="figA1020" rid-ob="figobA1020">Figure 6-29</a>), is composed of one RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> and at least seven proteins. The U6 <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5812/" role="button" aria-expanded="false" aria-haspopup="true">snRNA</a> is chemically modified by snoRNAs in the nucleolus before its final assembly there into the U6 snRNP. Other important RNA protein complexes, including <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5858/" role="button" aria-expanded="false" aria-haspopup="true">telomerase</a> (encountered in Chapter 5) and the signal recognition particle (which we discuss in Chapter 12), are also believed to be assembled at the nucleolus. Finally, the tRNAs (transfer RNAs) that carry the amino acids for protein synthesis are processed there as well. Thus, the nucleolus can be thought of as a large factory at which many different noncoding RNAs are processed and assembled with proteins to form a large variety of ribonucleoprotein complexes.

</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1047" co-legend-rid="figlgndA1047"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1047/?report=objectonly" target="object" title="Figure 6-47" class="img_link icnblk_img figpopup" rid-figpopup="figA1047" rid-ob="figobA1047"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f47.gif" src-large="/books/NBK26887/bin/ch6f47.jpg" alt="Figure 6-47. The function of the nucleolus in ribosome and other ribonucleoprotein synthesis."></a><div class="icnblk_cntnt" id="figlgndA1047"><h4 id="A1047"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1047/?report=objectonly" target="object" rid-ob="figobA1047">Figure 6-47</a></h4><p class="float-caption no_bottom_margin">The function of the nucleolus in ribosome and other ribonucleoprotein synthesis. The 45S precursor rRNA is packaged in a large ribonucleoprotein particle containing many ribosomal proteins imported from the cytoplasm. While this particle remains in the <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1047/?report=objectonly" target="object" rid-ob="figobA1047">(more...)</a></p></div></div></div><div id="A1048"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1048_" class="ui-helper-clearfix">The Nucleus Contains a Variety of Subnuclear Structures</h2><p>Although the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a> is the most prominent structure in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>, several other nuclear bodies have been visualized and studied (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1049/?report=objectonly" target="object" rid-figpopup="figA1049" rid-ob="figobA1049">Figure 6-48</a>). These include Cajal bodies (named for the scientist who first described them in 1906), GEMS (Gemini of coiled bodies), and interchromatin granule clusters (also called “speckles”). Like the nucleolus, these other nuclear structures lack membranes and are highly dynamic; their appearance is probably the result of the tight association of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> (and perhaps <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a>) components involved in the synthesis, assembly, and storage of macromolecules involved in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5163/" role="button" aria-expanded="false" aria-haspopup="true">expression</a>. Cajal bodies and GEMS resemble one another and are frequently paired in the nucleus; it is not clear whether they are truly distinct structures. They may be sites where snRNAs and snoRNAs undergo their final modifications and assembly with protein. Both the RNAs and the proteins that make up the snRNPs are partly assembled in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5053/" role="button" aria-expanded="false" aria-haspopup="true">cytoplasm</a>, but they are transported into the nucleus for their final modifications. It has been proposed that Cajal bodies/GEMS are also sites where the snRNPs are recycled and their RNAs are “reset” after the rearrangements that occur during splicing (see p. 322). In contrast, the interchromatin granule clusters have been proposed to be stockpiles of fully mature snRNPs that are ready to be used in splicing of pre-mRNAs (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1050/?report=objectonly" target="object" rid-figpopup="figA1050" rid-ob="figobA1050">Figure 6-49</a>).


</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1049" co-legend-rid="figlgndA1049"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1049/?report=objectonly" target="object" title="Figure 6-48" class="img_link icnblk_img figpopup" rid-figpopup="figA1049" rid-ob="figobA1049"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f48.gif" src-large="/books/NBK26887/bin/ch6f48.jpg" alt="Figure 6-48. Visualization of chromatin and nuclear bodies."></a><div class="icnblk_cntnt" id="figlgndA1049"><h4 id="A1049"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1049/?report=objectonly" target="object" rid-ob="figobA1049">Figure 6-48</a></h4><p class="float-caption no_bottom_margin">Visualization of chromatin and nuclear bodies. (A)-(D) show micrographs of the same human cell nucleus, each processed differently to show a particular set of nuclear structures. (E) shows an enlarged superposition of all four individual images. (A) shows <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1049/?report=objectonly" target="object" rid-ob="figobA1049">(more...)</a></p></div></div><div class="iconblock whole_rhythm clearfix ten_col fig" id="figA1050" co-legend-rid="figlgndA1050"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1050/?report=objectonly" target="object" title="Figure 6-49" class="img_link icnblk_img figpopup" rid-figpopup="figA1050" rid-ob="figobA1050"><img class="small-thumb" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/ch6f49.gif" src-large="/books/NBK26887/bin/ch6f49.jpg" alt="Figure 6-49. Schematic view of subnuclear structures."></a><div class="icnblk_cntnt" id="figlgndA1050"><h4 id="A1050"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1050/?report=objectonly" target="object" rid-ob="figobA1050">Figure 6-49</a></h4><p class="float-caption no_bottom_margin">Schematic view of subnuclear structures. A typical vertebrate nucleus has several Cajal bodies, which are proposed to be the sites where snRNPs and snoRNPs undergo their final modifications. Interchromatin granule clusters are proposed to be storage sites <a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1050/?report=objectonly" target="object" rid-ob="figobA1050">(more...)</a></p></div></div><p>Scientists have had difficulties in working out the function of the small subnuclear structures just described. Much of the progress now being made depends on genetic tools—examination of the effects of designed mutations in mice or of spontaneous mutations in humans. As one example, GEMS contain the SMN (survival of motor neurons) <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a>. Certain mutations of the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> encoding this protein are the cause of inherited spinal muscular atrophy, a human disease characterized by a wasting away of the muscles. The disease seems to be caused by a subtle defect in snRNP assembly and subsequent pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing. More severe defects would be expected to be lethal.</p><p>Given the importance of nuclear subdomains in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> processing, it might have been expected that pre-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> splicing would occur in a particular location in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/" role="button" aria-expanded="false" aria-haspopup="true">nucleus</a>, as it requires numerous RNA and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> components. However, we have seen that the assembly of splicing components on pre-mRNA is co-transcriptional; thus splicing must occur at many locations along chromosomes. We saw in Chapter 4 that <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5350/" role="button" aria-expanded="false" aria-haspopup="true">interphase</a> chromosomes occupy discrete territories in the nucleus, and transcription and pre-mRNA splicing must take place within these territories. However, interphase chromosomes are themselves dynamic and their exact positioning in the nucleus correlates with <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/" role="button" aria-expanded="false" aria-haspopup="true">gene</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5163/" role="button" aria-expanded="false" aria-haspopup="true">expression</a>. For example, transcriptionally silent regions of interphase chromosomes are often associated with the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5551/" role="button" aria-expanded="false" aria-haspopup="true">nuclear envelope</a> where the concentration of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5276/" role="button" aria-expanded="false" aria-haspopup="true">heterochromatin</a> components is believed to be especially high. When these same regions become transcriptionally active, they relocate towards the interior of the nucleus, which is richer in the components required for mRNA synthesis. It has been proposed that, although a typical mammalian cell may be expressing on the order of 15,000 genes, transcription and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a> may be localized to only several thousand sites in the nucleus. These sites themselves are highly dynamic and probably result from the association of transcription and splicing components to create small “assembly lines” where the local concentration of these components is very high. As a result, the nucleus seems to be highly organized into subdomains, with snRNPs, snoRNPs, and other nuclear components moving between them in an orderly fashion according to the needs of the cell (<a class="figpopup" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1050/?report=objectonly" target="object" rid-figpopup="figA1050" rid-ob="figobA1050">Figure 6-49</a>).</p></div><div id="A1051"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 id="_A1051_" class="ui-helper-clearfix">Summary</h2><p>Before the synthesis of a particular <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> can begin, the corresponding <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> must be produced by transcription. Bacteria contain a single type of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> (the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> that carries out the transcription of <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> into RNA). An mRNA molecule is produced when this enzyme initiates transcription at a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/" role="button" aria-expanded="false" aria-haspopup="true">promoter</a>, synthesizes the RNA by chain elongation, stops transcription at a <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5863/" role="button" aria-expanded="false" aria-haspopup="true">terminator</a>, and releases both the DNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> and the completed mRNA molecule. In eucaryotic cells, the process of transcription is much more <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/" role="button" aria-expanded="false" aria-haspopup="true">complex</a>, and there are three RNA polymerases—designated polymerase I, II, and III—that are related evolutionarily to one another and to the bacterial polymerase.</p><p>Eucaryotic <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/" role="button" aria-expanded="false" aria-haspopup="true">mRNA</a> is synthesized by <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> II. This <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/" role="button" aria-expanded="false" aria-haspopup="true">enzyme</a> requires a series of additional proteins, termed the general transcription factors, to initiate transcription on a purified <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/" role="button" aria-expanded="false" aria-haspopup="true">DNA</a> <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/" role="button" aria-expanded="false" aria-haspopup="true">template</a> and still more proteins (including <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4974/" role="button" aria-expanded="false" aria-haspopup="true">chromatin</a>-remodeling complexes and <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5283/" role="button" aria-expanded="false" aria-haspopup="true">histone</a> acetyltransferases) to initiate transcription on its chromatin template inside the cell. During the elongation phase of transcription, the nascent RNA undergoes three types of processing events: a special <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/" role="button" aria-expanded="false" aria-haspopup="true">nucleotide</a> is added to its 5′ end (capping), <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/" role="button" aria-expanded="false" aria-haspopup="true">intron</a> sequences are removed from the middle of the RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/" role="button" aria-expanded="false" aria-haspopup="true">molecule</a> (splicing), and the 3′ end of the RNA is generated (<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/" role="button" aria-expanded="false" aria-haspopup="true">cleavage</a> and polyadenylation). Some of these RNA processing events that modify the initial RNA <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/" role="button" aria-expanded="false" aria-haspopup="true">transcript</a> (for example, those involved in <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/" role="button" aria-expanded="false" aria-haspopup="true">RNA splicing</a>) are carried out primarily by special small RNA molecules.</p><p>For some genes, <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/" role="button" aria-expanded="false" aria-haspopup="true">RNA</a> is the final product. In eucaryotes, these genes are usually transcribed by either <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/" role="button" aria-expanded="false" aria-haspopup="true">RNA polymerase</a> I or RNA polymerase III. RNA polymerase I makes the ribosomal RNAs. After their synthesis as a large precursor, the rRNAs are chemically modified, cleaved, and assembled into ribosomes in the <a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/" role="button" aria-expanded="false" aria-haspopup="true">nucleolus</a>—a distinct subnuclear structure that also helps to process some smaller RNA-<a class="def" href="http://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/" role="button" aria-expanded="false" aria-haspopup="true">protein</a> complexes in the cell. Additional subnuclear structures (including Cajal bodies and interchromatin granule clusters) are sites where components involved in RNA processing are assembled, stored, and recycled.</p></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-1"><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A977_">Portions of DNA Sequence Are Transcribed into RNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A981_">Transcription Produces RNA Complementary to One Strand of DNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A985_">Cells Produce Several Types of RNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A987_">Signals Encoded in DNA Tell RNA Polymerase Where to Start and Stop</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A990_">Transcription Start and Stop Signals Are Heterogeneous in Nucleotide Sequence</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A994_">Transcription Initiation in Eucaryotes Requires Many Proteins</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A999_">RNA Polymerase II Requires General Transcription Factors</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1003_">Polymerase II Also Requires Activator, Mediator, and Chromatin-modifying Proteins</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1005_">Transcription Elongation Produces Superhelical Tension in DNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1007_">Transcription Elongation in Eucaryotes Is Tightly Coupled To RNA Processing</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1011_">RNA Capping Is the First Modification of Eucaryotic Pre-mRNAs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1013_">RNA Splicing Removes Intron Sequences from Newly Transcribed Pre-mRNAs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1017_">Nucleotide Sequences Signal Where Splicing Occurs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1019_">RNA Splicing Is Performed by the Spliceosome</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1022_">The Spliceosome Uses ATP Hydrolysis to Produce a Complex Series of RNA-RNA Rearrangements</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1023_">Ordering Influences in the Pre-mRNA Help to Explain How the Proper Splice Sites Are Chosen</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1027_">A Second Set of snRNPs Splice a Small Fraction of Intron Sequences in Animals and Plants</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1029_">RNA Splicing Shows Remarkable Plasticity</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1031_">Spliceosome-catalyzed RNA Splicing Probably Evolved from Self-splicing Mechanisms</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1033_">RNA-Processing Enzymes Generate the 3′ End of Eucaryotic mRNAs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1036_">Mature Eucaryotic mRNAs Are Selectively Exported from the Nucleus</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1039_">Many Noncoding RNAs Are Also Synthesized and Processed in the Nucleus</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1043_">The Nucleolus Is a Ribosome-Producing Factory</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1048_">The Nucleus Contains a Variety of Subnuclear Structures</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_A1051_">Summary</a></li></ul></div></div></div>
            <div class="post-content"><div><p>By agreement with the publisher, this book is accessible by the search feature, but cannot be browsed.</p><div class="half_rhythm"><a href="http://www.ncbi.nlm.nih.gov/books/about/copyright/">Copyright</a> © 2002, Bruce Alberts, Alexander Johnson, Julian
Lewis, Martin Raff, Keith Roberts, and Peter Walter; Copyright © 1983,
1989, 1994, Bruce Alberts, Dennis Bray, Julian Lewis, Martin Raff, Keith Roberts,
and James D. Watson .</div><div class="small"><span class="label">Bookshelf ID: NBK26887</span></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        <div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter" toggles="ui-portlet_content-2" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-2" aria-hidden="false"><ul><li><a class="jig-ncbidialog" data-jigconfig="width:400 , modal:true" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#_ncbi_dlg_citbx_NBK26887">Cite this Page</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this Page</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter" toggles="ui-portlet_content-3" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-3" aria-hidden="false"><ul><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A977">Portions of DNA Sequence Are Transcribed into RNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A981">Transcription Produces RNA Complementary to One Strand of DNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A985">Cells Produce Several Types of RNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A987">Signals Encoded in DNA Tell RNA Polymerase Where to Start and Stop</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A990">Transcription Start and Stop Signals Are Heterogeneous in Nucleotide Sequence</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A994">Transcription Initiation in Eucaryotes Requires Many Proteins</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A999">RNA Polymerase II Requires General Transcription Factors</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1003">Polymerase II Also Requires Activator, Mediator, and Chromatin-modifying Proteins</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1005">Transcription Elongation Produces Superhelical Tension in DNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1007">Transcription Elongation in Eucaryotes Is Tightly Coupled To RNA Processing</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1011">RNA Capping Is the First Modification of Eucaryotic Pre-mRNAs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1013">RNA Splicing Removes Intron Sequences from Newly Transcribed Pre-mRNAs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1017">Nucleotide Sequences Signal Where Splicing Occurs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1019">RNA Splicing Is Performed by the Spliceosome</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1022">The Spliceosome Uses ATP Hydrolysis to Produce a Complex Series of RNA-RNA Rearrangements</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1023">Ordering Influences in the Pre-mRNA Help to Explain How the Proper Splice Sites Are Chosen</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1027">A Second Set of snRNPs Splice a Small Fraction of Intron Sequences in Animals and Plants</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1029">RNA Splicing Shows Remarkable Plasticity</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1031">Spliceosome-catalyzed RNA Splicing Probably Evolved from Self-splicing Mechanisms</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1033">RNA-Processing Enzymes Generate the 3′ End of Eucaryotic mRNAs</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1036">Mature Eucaryotic mRNAs Are Selectively Exported from the Nucleus</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1039">Many Noncoding RNAs Are Also Synthesized and Processed in the Nucleus</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1043">The Nucleolus Is a Ribosome-Producing Factory</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1048">The Nucleus Contains a Variety of Subnuclear Structures</a></li><li><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#A1051">Summary</a></li></ul></div></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter" toggles="ui-portlet_content-4" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line" role="button" aria-expanded="false" aria-haspopup="true"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="http://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=1376640602109132">From DNA to RNA - Molecular Biology of the Cell</a><div class="ralinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-ralinkpop-5" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">From DNA to RNA - Molecular Biology of the Cell<div class="brieflinkpopdesc"></div></div><div class="tertiary">Bookshelf</div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="http://www.ncbi.nlm.nih.gov/sites/myncbi/recentactivity">See more...</a></div></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="http://www.ncbi.nlm.nih.gov/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/books">Bookshelf</a></span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="http://www.ncbi.nlm.nih.gov/sites/ehelp?&Ncbi_App=bookshelf&Db=books&Page=literature&Snapshot=/projects/books/PBooks@4.12&Time=2013-08-16T04:10:02-04:00&Host=portal206&key=1ZUl6mlwh8sgqqpBjg6-LLck-">Write to the Help Desk</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Copyright</a> | <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="http://www.ncbi.nlm.nih.gov/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="http://www.ncbi.nlm.nih.gov/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/InstrumentPageStarterJS.js"> </script>    
    <script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/hfjs.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow" aria-live="assertive"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant=""></ul><div class="ui-ncbiautocomplete-actions shadow"><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="ui-ncbiautocomplete-link-pref">Preferences</a><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="ui-ncbiautocomplete-link-off">Turn off</a></div></div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=books&amp;amp;ncbi_pdid=book-part&amp;amp;ncbi_acc=NBK26887&amp;amp;ncbi_domain=mboc4&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/NBK26887/&amp;amp;ncbi_app=bookshelf" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- 396A777215AA0C71_0094SID /projects/books/PBooks@4.12 portal206 v3.5.1.r392364: Mon, Mar 25 2013 15:07:09 -->

<script type="text/javascript" src="./From DNA to RNA - Molecular Biology of the Cell - NCBI Bookshelf_files/3426610.js" snapshot="books"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all" tabindex="-1" role="dialog" aria-labelledby="ui-dialog-title-_ncbi_dlg_citbx_NBK26887" style="display: none; z-index: 1000; outline: 0px;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span class="ui-dialog-title" id="ui-dialog-title-_ncbi_dlg_citbx_NBK26887">Cite this Page</span><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="_ncbi_dlg_citbx_NBK26887" class="jig-ncbidialog-box ui-dialog-content ui-widget-content" style=""><div class="bk_tt">Alberts B, Johnson A, Lewis J, et al. Molecular Biology of the Cell. 4th edition. New York: Garland Science; 2002.  From DNA to RNA.<span class="bk_cite_avail"> Available from: http://www.ncbi.nlm.nih.gov/books/NBK26887/</span></div></div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all" tabindex="-1" role="dialog" aria-labelledby="ui-dialog-title-epubDialog" style="display: none; z-index: 1000; outline: 0px;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span class="ui-dialog-title" id="ui-dialog-title-epubDialog">Making content easier to read in Bookshelf</span><a href="http://www.ncbi.nlm.nih.gov/books/NBK26887/#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="epubDialog" style="" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read books and documents     in Bookshelf.     Our first effort uses ebook readers, which have several "ease of reading"     features already built in.</p>  <p>The content is best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of some features of books or documents    in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download</button></div>          </div></div><ul class="print-log"><li></li></ul></body></html>